<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type">
          <link rel="stylesheet" href="https://unpkg.com/higlass@0.8.8/dist/styles/hglib.css" type="text/css">


<style type="text/css">
body {
    line-height: 1.6;
}

.lst-kix_g7gt31185t1l-7>li:before {
    content: "-  "
}
.lst-kix_ao7ckee9pbpp-6>li {
    counter-increment: lst-ctn-kix_ao7ckee9pbpp-6
}
ol.lst-kix_k2d1kb8p2zv5-5.start {
    counter-reset: lst-ctn-kix_k2d1kb8p2zv5-5 0
}
.lst-kix_g7gt31185t1l-8>li:before {
    content: "-  "
}
.lst-kix_lqlac0obey9y-8>li {
    counter-increment: lst-ctn-kix_lqlac0obey9y-8
}
.lst-kix_lqlac0obey9y-2>li:before {
    content: "" counter(lst-ctn-kix_lqlac0obey9y-2, lower-roman) ". "
}
.lst-kix_g7gt31185t1l-3>li:before {
    content: "-  "
}
.lst-kix_lqlac0obey9y-3>li:before {
    content: "" counter(lst-ctn-kix_lqlac0obey9y-3, decimal) ". "
}
.lst-kix_g7gt31185t1l-2>li:before {
    content: "-  "
}
.lst-kix_lqlac0obey9y-5>li:before {
    content: "" counter(lst-ctn-kix_lqlac0obey9y-5, lower-roman) ". "
}
.lst-kix_lqlac0obey9y-4>li:before {
    content: "" counter(lst-ctn-kix_lqlac0obey9y-4, lower-latin) ". "
}
.lst-kix_lqlac0obey9y-6>li:before {
    content: "" counter(lst-ctn-kix_lqlac0obey9y-6, decimal) ". "
}
ol.lst-kix_ao7ckee9pbpp-6.start {
    counter-reset: lst-ctn-kix_ao7ckee9pbpp-6 0
}
.lst-kix_g7gt31185t1l-4>li:before {
    content: "-  "
}
.lst-kix_lqlac0obey9y-8>li:before {
    content: "" counter(lst-ctn-kix_lqlac0obey9y-8, lower-roman) ". "
}
.lst-kix_g7gt31185t1l-5>li:before {
    content: "-  "
}
.lst-kix_lqlac0obey9y-7>li:before {
    content: "" counter(lst-ctn-kix_lqlac0obey9y-7, lower-latin) ". "
}
.lst-kix_g7gt31185t1l-6>li:before {
    content: "-  "
}
.lst-kix_905ht9yftleh-0>li:before {
    content: "\0025cf  "
}
.lst-kix_905ht9yftleh-2>li:before {
    content: "\0025a0  "
}
.lst-kix_905ht9yftleh-3>li:before {
    content: "\0025cf  "
}
.lst-kix_k2d1kb8p2zv5-6>li:before {
    content: "" counter(lst-ctn-kix_k2d1kb8p2zv5-6, decimal) ". "
}
.lst-kix_905ht9yftleh-1>li:before {
    content: "\0025cb  "
}
.lst-kix_k2d1kb8p2zv5-7>li:before {
    content: "" counter(lst-ctn-kix_k2d1kb8p2zv5-7, lower-latin) ". "
}
.lst-kix_905ht9yftleh-8>li:before {
    content: "\0025a0  "
}
.lst-kix_k2d1kb8p2zv5-8>li:before {
    content: "" counter(lst-ctn-kix_k2d1kb8p2zv5-8, lower-roman) ". "
}
.lst-kix_905ht9yftleh-7>li:before {
    content: "\0025cb  "
}
.lst-kix_905ht9yftleh-4>li:before {
    content: "\0025cb  "
}
.lst-kix_905ht9yftleh-6>li:before {
    content: "\0025cf  "
}
.lst-kix_905ht9yftleh-5>li:before {
    content: "\0025a0  "
}
ul.lst-kix_x7qfbxbrd6e4-3 {
    list-style-type: none
}
ul.lst-kix_x7qfbxbrd6e4-2 {
    list-style-type: none
}
ul.lst-kix_x7qfbxbrd6e4-5 {
    list-style-type: none
}
ul.lst-kix_x7qfbxbrd6e4-4 {
    list-style-type: none
}
ul.lst-kix_x7qfbxbrd6e4-1 {
    list-style-type: none
}
ul.lst-kix_x7qfbxbrd6e4-0 {
    list-style-type: none
}
.lst-kix_k2d1kb8p2zv5-2>li {
    counter-increment: lst-ctn-kix_k2d1kb8p2zv5-2
}
.lst-kix_lqlac0obey9y-4>li {
    counter-increment: lst-ctn-kix_lqlac0obey9y-4
}
ol.lst-kix_ao7ckee9pbpp-1.start {
    counter-reset: lst-ctn-kix_ao7ckee9pbpp-1 0
}
.lst-kix_okc0u988m2rc-8>li:before {
    content: "\0025a0  "
}
ul.lst-kix_egnx6h47ljtr-3 {
    list-style-type: none
}
ul.lst-kix_egnx6h47ljtr-2 {
    list-style-type: none
}
.lst-kix_okc0u988m2rc-5>li:before {
    content: "\0025a0  "
}
ul.lst-kix_secj84g03ko5-0 {
    list-style-type: none
}
ul.lst-kix_egnx6h47ljtr-1 {
    list-style-type: none
}
ul.lst-kix_egnx6h47ljtr-0 {
    list-style-type: none
}
ul.lst-kix_secj84g03ko5-3 {
    list-style-type: none
}
.lst-kix_okc0u988m2rc-6>li:before {
    content: "\0025cf  "
}
.lst-kix_okc0u988m2rc-7>li:before {
    content: "\0025cb  "
}
ul.lst-kix_secj84g03ko5-4 {
    list-style-type: none
}
ul.lst-kix_secj84g03ko5-1 {
    list-style-type: none
}
ul.lst-kix_secj84g03ko5-2 {
    list-style-type: none
}
.lst-kix_okc0u988m2rc-0>li:before {
    content: "\0025cf  "
}
.lst-kix_okc0u988m2rc-1>li:before {
    content: "\0025cb  "
}
ol.lst-kix_lqlac0obey9y-5.start {
    counter-reset: lst-ctn-kix_lqlac0obey9y-5 0
}
.lst-kix_okc0u988m2rc-4>li:before {
    content: "\0025cb  "
}
ul.lst-kix_x7qfbxbrd6e4-7 {
    list-style-type: none
}
.lst-kix_okc0u988m2rc-2>li:before {
    content: "\0025a0  "
}
.lst-kix_okc0u988m2rc-3>li:before {
    content: "\0025cf  "
}
ul.lst-kix_x7qfbxbrd6e4-6 {
    list-style-type: none
}
ul.lst-kix_x7qfbxbrd6e4-8 {
    list-style-type: none
}
ul.lst-kix_okc0u988m2rc-3 {
    list-style-type: none
}
ul.lst-kix_55qcczpq6rq9-1 {
    list-style-type: none
}
.lst-kix_x0bujimllc2-3>li:before {
    content: "\0025cf  "
}
ul.lst-kix_okc0u988m2rc-2 {
    list-style-type: none
}
ul.lst-kix_55qcczpq6rq9-2 {
    list-style-type: none
}
ul.lst-kix_okc0u988m2rc-5 {
    list-style-type: none
}
.lst-kix_k2d1kb8p2zv5-5>li {
    counter-increment: lst-ctn-kix_k2d1kb8p2zv5-5
}
.lst-kix_x0bujimllc2-4>li:before {
    content: "\0025cb  "
}
ul.lst-kix_okc0u988m2rc-4 {
    list-style-type: none
}
ul.lst-kix_55qcczpq6rq9-0 {
    list-style-type: none
}
.lst-kix_x0bujimllc2-1>li:before {
    content: "\0025cb  "
}
ul.lst-kix_okc0u988m2rc-1 {
    list-style-type: none
}
ul.lst-kix_okc0u988m2rc-0 {
    list-style-type: none
}
ol.lst-kix_lqlac0obey9y-0.start {
    counter-reset: lst-ctn-kix_lqlac0obey9y-0 0
}
.lst-kix_x0bujimllc2-6>li:before {
    content: "\0025cf  "
}
.lst-kix_x0bujimllc2-8>li:before {
    content: "\0025a0  "
}
ul.lst-kix_okc0u988m2rc-7 {
    list-style-type: none
}
ul.lst-kix_okc0u988m2rc-6 {
    list-style-type: none
}
.lst-kix_osealum981ss-3>li:before {
    content: "\0025cf  "
}
ul.lst-kix_okc0u988m2rc-8 {
    list-style-type: none
}
ul.lst-kix_secj84g03ko5-7 {
    list-style-type: none
}
ul.lst-kix_secj84g03ko5-8 {
    list-style-type: none
}
.lst-kix_osealum981ss-5>li:before {
    content: "\0025a0  "
}
ul.lst-kix_905ht9yftleh-0 {
    list-style-type: none
}
ul.lst-kix_secj84g03ko5-5 {
    list-style-type: none
}
ul.lst-kix_egnx6h47ljtr-8 {
    list-style-type: none
}
.lst-kix_lqlac0obey9y-1>li {
    counter-increment: lst-ctn-kix_lqlac0obey9y-1
}
ul.lst-kix_905ht9yftleh-1 {
    list-style-type: none
}
ul.lst-kix_secj84g03ko5-6 {
    list-style-type: none
}
ul.lst-kix_egnx6h47ljtr-7 {
    list-style-type: none
}
ul.lst-kix_egnx6h47ljtr-6 {
    list-style-type: none
}
ul.lst-kix_egnx6h47ljtr-5 {
    list-style-type: none
}
ul.lst-kix_egnx6h47ljtr-4 {
    list-style-type: none
}
ul.lst-kix_or8pcwk62bfz-5 {
    list-style-type: none
}
ul.lst-kix_905ht9yftleh-6 {
    list-style-type: none
}
ul.lst-kix_or8pcwk62bfz-6 {
    list-style-type: none
}
ul.lst-kix_905ht9yftleh-7 {
    list-style-type: none
}
ul.lst-kix_or8pcwk62bfz-3 {
    list-style-type: none
}
ul.lst-kix_55qcczpq6rq9-7 {
    list-style-type: none
}
ul.lst-kix_905ht9yftleh-8 {
    list-style-type: none
}
ul.lst-kix_or8pcwk62bfz-4 {
    list-style-type: none
}
ul.lst-kix_55qcczpq6rq9-8 {
    list-style-type: none
}
ol.lst-kix_lqlac0obey9y-3.start {
    counter-reset: lst-ctn-kix_lqlac0obey9y-3 0
}
ul.lst-kix_55qcczpq6rq9-5 {
    list-style-type: none
}
ul.lst-kix_905ht9yftleh-2 {
    list-style-type: none
}
ul.lst-kix_55qcczpq6rq9-6 {
    list-style-type: none
}
ul.lst-kix_905ht9yftleh-3 {
    list-style-type: none
}
.lst-kix_osealum981ss-7>li:before {
    content: "\0025cb  "
}
ul.lst-kix_or8pcwk62bfz-7 {
    list-style-type: none
}
ul.lst-kix_55qcczpq6rq9-3 {
    list-style-type: none
}
ul.lst-kix_905ht9yftleh-4 {
    list-style-type: none
}
ul.lst-kix_or8pcwk62bfz-8 {
    list-style-type: none
}
ul.lst-kix_55qcczpq6rq9-4 {
    list-style-type: none
}
ul.lst-kix_905ht9yftleh-5 {
    list-style-type: none
}
.lst-kix_ao7ckee9pbpp-7>li:before {
    content: "" counter(lst-ctn-kix_ao7ckee9pbpp-7, lower-latin) ". "
}
ul.lst-kix_or8pcwk62bfz-1 {
    list-style-type: none
}
.lst-kix_ao7ckee9pbpp-5>li:before {
    content: "" counter(lst-ctn-kix_ao7ckee9pbpp-5, lower-roman) ". "
}
ul.lst-kix_or8pcwk62bfz-2 {
    list-style-type: none
}
ul.lst-kix_or8pcwk62bfz-0 {
    list-style-type: none
}
.lst-kix_ao7ckee9pbpp-3>li:before {
    content: "" counter(lst-ctn-kix_ao7ckee9pbpp-3, decimal) ". "
}
.lst-kix_g7ryix1tcu3y-0>li:before {
    content: "\0025cf  "
}
.lst-kix_ao4h9rgb0ekx-8>li:before {
    content: "\0025a0  "
}
.lst-kix_2qrmnwynf1xh-0>li:before {
    content: "\0025cf  "
}
.lst-kix_xckeupt8b5jl-4>li:before {
    content: "\0025cb  "
}
.lst-kix_2qrmnwynf1xh-2>li:before {
    content: "\0025a0  "
}
.lst-kix_2qrmnwynf1xh-6>li:before {
    content: "\0025cf  "
}
.lst-kix_g7ryix1tcu3y-6>li:before {
    content: "\0025cf  "
}
.lst-kix_xckeupt8b5jl-8>li:before {
    content: "\0025a0  "
}
.lst-kix_xckeupt8b5jl-0>li:before {
    content: "\0025cf  "
}
.lst-kix_xckeupt8b5jl-2>li:before {
    content: "\0025a0  "
}
.lst-kix_2qrmnwynf1xh-4>li:before {
    content: "\0025cb  "
}
.lst-kix_g7ryix1tcu3y-8>li:before {
    content: "\0025a0  "
}
ul.lst-kix_aeeme22hncrc-5 {
    list-style-type: none
}
.lst-kix_g7ryix1tcu3y-2>li:before {
    content: "\0025a0  "
}
ul.lst-kix_aeeme22hncrc-4 {
    list-style-type: none
}
ul.lst-kix_aeeme22hncrc-7 {
    list-style-type: none
}
ul.lst-kix_aeeme22hncrc-6 {
    list-style-type: none
}
.lst-kix_ao7ckee9pbpp-1>li:before {
    content: "" counter(lst-ctn-kix_ao7ckee9pbpp-1, lower-latin) ". "
}
ul.lst-kix_aeeme22hncrc-8 {
    list-style-type: none
}
.lst-kix_k2d1kb8p2zv5-4>li:before {
    content: "" counter(lst-ctn-kix_k2d1kb8p2zv5-4, lower-latin) ". "
}
ul.lst-kix_aeeme22hncrc-1 {
    list-style-type: none
}
.lst-kix_h3upo6j1qfko-2>li:before {
    content: "\0025a0  "
}
.lst-kix_xckeupt8b5jl-6>li:before {
    content: "\0025cf  "
}
ul.lst-kix_aeeme22hncrc-0 {
    list-style-type: none
}
ul.lst-kix_aeeme22hncrc-3 {
    list-style-type: none
}
.lst-kix_2qrmnwynf1xh-8>li:before {
    content: "\0025a0  "
}
.lst-kix_k2d1kb8p2zv5-2>li:before {
    content: "" counter(lst-ctn-kix_k2d1kb8p2zv5-2, lower-roman) ". "
}
ul.lst-kix_aeeme22hncrc-2 {
    list-style-type: none
}
.lst-kix_g7ryix1tcu3y-4>li:before {
    content: "\0025cb  "
}
.lst-kix_h3upo6j1qfko-0>li:before {
    content: "\0025cf  "
}
.lst-kix_k2d1kb8p2zv5-0>li:before {
    content: "" counter(lst-ctn-kix_k2d1kb8p2zv5-0, decimal) ". "
}
.lst-kix_lqlac0obey9y-0>li {
    counter-increment: lst-ctn-kix_lqlac0obey9y-0
}
.lst-kix_ao7ckee9pbpp-2>li {
    counter-increment: lst-ctn-kix_ao7ckee9pbpp-2
}
.lst-kix_g7gt31185t1l-0>li:before {
    content: "-  "
}
.lst-kix_lqlac0obey9y-1>li:before {
    content: "" counter(lst-ctn-kix_lqlac0obey9y-1, lower-latin) ". "
}
ol.lst-kix_k2d1kb8p2zv5-7.start {
    counter-reset: lst-ctn-kix_k2d1kb8p2zv5-7 0
}
.lst-kix_osealum981ss-1>li:before {
    content: "\0025cb  "
}
ul.lst-kix_v6ecskul40c2-0 {
    list-style-type: none
}
ul.lst-kix_ao4h9rgb0ekx-0 {
    list-style-type: none
}
ul.lst-kix_v6ecskul40c2-7 {
    list-style-type: none
}
ul.lst-kix_ao4h9rgb0ekx-2 {
    list-style-type: none
}
ul.lst-kix_v6ecskul40c2-8 {
    list-style-type: none
}
ul.lst-kix_ao4h9rgb0ekx-1 {
    list-style-type: none
}
ul.lst-kix_v6ecskul40c2-5 {
    list-style-type: none
}
ul.lst-kix_ao4h9rgb0ekx-4 {
    list-style-type: none
}
ul.lst-kix_v6ecskul40c2-6 {
    list-style-type: none
}
ul.lst-kix_ao4h9rgb0ekx-3 {
    list-style-type: none
}
ul.lst-kix_v6ecskul40c2-3 {
    list-style-type: none
}
ul.lst-kix_ao4h9rgb0ekx-6 {
    list-style-type: none
}
.lst-kix_or8pcwk62bfz-1>li:before {
    content: "-  "
}
ul.lst-kix_v6ecskul40c2-4 {
    list-style-type: none
}
ul.lst-kix_ao4h9rgb0ekx-5 {
    list-style-type: none
}
.lst-kix_z7nrbovt5e2k-7>li:before {
    content: "\0025cb  "
}
.lst-kix_or8pcwk62bfz-8>li:before {
    content: "-  "
}
.lst-kix_x7qfbxbrd6e4-3>li:before {
    content: "-  "
}
ul.lst-kix_v6ecskul40c2-1 {
    list-style-type: none
}
ul.lst-kix_ao4h9rgb0ekx-8 {
    list-style-type: none
}
ul.lst-kix_v6ecskul40c2-2 {
    list-style-type: none
}
ul.lst-kix_ao4h9rgb0ekx-7 {
    list-style-type: none
}
.lst-kix_bde8pay3hzip-1>li:before {
    content: "\0025cb  "
}
.lst-kix_rhatjkb0huf0-1>li:before {
    content: "\0025cb  "
}
.lst-kix_rhatjkb0huf0-2>li:before {
    content: "\0025a0  "
}
.lst-kix_x7qfbxbrd6e4-6>li:before {
    content: "-  "
}
.lst-kix_or8pcwk62bfz-5>li:before {
    content: "-  "
}
.lst-kix_bde8pay3hzip-0>li:before {
    content: "\0025cf  "
}
.lst-kix_or8pcwk62bfz-4>li:before {
    content: "-  "
}
.lst-kix_x7qfbxbrd6e4-7>li:before {
    content: "-  "
}
.lst-kix_rhatjkb0huf0-5>li:before {
    content: "\0025a0  "
}
.lst-kix_rhatjkb0huf0-6>li:before {
    content: "\0025cf  "
}
.lst-kix_h3upo6j1qfko-4>li:before {
    content: "\0025cb  "
}
.lst-kix_ao7ckee9pbpp-7>li {
    counter-increment: lst-ctn-kix_ao7ckee9pbpp-7
}
.lst-kix_secj84g03ko5-5>li:before {
    content: "-  "
}
.lst-kix_h3upo6j1qfko-5>li:before {
    content: "\0025a0  "
}
ul.lst-kix_z7nrbovt5e2k-0 {
    list-style-type: none
}
.lst-kix_h3upo6j1qfko-8>li:before {
    content: "\0025a0  "
}
.lst-kix_x7qfbxbrd6e4-2>li:before {
    content: "-  "
}
ul.lst-kix_so04ye1qhvjl-2 {
    list-style-type: none
}
ul.lst-kix_pd9b5gi6xvdb-6 {
    list-style-type: none
}
ul.lst-kix_z7nrbovt5e2k-1 {
    list-style-type: none
}
.lst-kix_or8pcwk62bfz-0>li:before {
    content: "-  "
}
ul.lst-kix_so04ye1qhvjl-1 {
    list-style-type: none
}
ul.lst-kix_pd9b5gi6xvdb-5 {
    list-style-type: none
}
ul.lst-kix_z7nrbovt5e2k-2 {
    list-style-type: none
}
ul.lst-kix_so04ye1qhvjl-4 {
    list-style-type: none
}
ul.lst-kix_pd9b5gi6xvdb-8 {
    list-style-type: none
}
ul.lst-kix_z7nrbovt5e2k-3 {
    list-style-type: none
}
ul.lst-kix_so04ye1qhvjl-3 {
    list-style-type: none
}
ul.lst-kix_pd9b5gi6xvdb-7 {
    list-style-type: none
}
ul.lst-kix_pd9b5gi6xvdb-2 {
    list-style-type: none
}
ul.lst-kix_pd9b5gi6xvdb-1 {
    list-style-type: none
}
ul.lst-kix_so04ye1qhvjl-0 {
    list-style-type: none
}
ul.lst-kix_pd9b5gi6xvdb-4 {
    list-style-type: none
}
ul.lst-kix_pd9b5gi6xvdb-3 {
    list-style-type: none
}
ul.lst-kix_z7nrbovt5e2k-8 {
    list-style-type: none
}
.lst-kix_secj84g03ko5-2>li:before {
    content: "-  "
}
ul.lst-kix_pd9b5gi6xvdb-0 {
    list-style-type: none
}
ul.lst-kix_z7nrbovt5e2k-4 {
    list-style-type: none
}
ol.lst-kix_lqlac0obey9y-7.start {
    counter-reset: lst-ctn-kix_lqlac0obey9y-7 0
}
.lst-kix_secj84g03ko5-1>li:before {
    content: "-  "
}
ul.lst-kix_z7nrbovt5e2k-5 {
    list-style-type: none
}
ul.lst-kix_z7nrbovt5e2k-6 {
    list-style-type: none
}
ol.lst-kix_k2d1kb8p2zv5-2.start {
    counter-reset: lst-ctn-kix_k2d1kb8p2zv5-2 0
}
ul.lst-kix_z7nrbovt5e2k-7 {
    list-style-type: none
}
.lst-kix_9cyha3bjgec2-0>li:before {
    content: "\0025cf  "
}
.lst-kix_ao7ckee9pbpp-3>li {
    counter-increment: lst-ctn-kix_ao7ckee9pbpp-3
}
.lst-kix_9cyha3bjgec2-1>li:before {
    content: "\0025cb  "
}
.lst-kix_ao4h9rgb0ekx-1>li:before {
    content: "\0025cb  "
}
.lst-kix_ao4h9rgb0ekx-2>li:before {
    content: "\0025a0  "
}
.lst-kix_9cyha3bjgec2-4>li:before {
    content: "\0025cb  "
}
.lst-kix_lqlac0obey9y-5>li {
    counter-increment: lst-ctn-kix_lqlac0obey9y-5
}
.lst-kix_aeeme22hncrc-2>li:before {
    content: "\0025a0  "
}
.lst-kix_aeeme22hncrc-1>li:before {
    content: "\0025cb  "
}
.lst-kix_secj84g03ko5-6>li:before {
    content: "-  "
}
ol.lst-kix_k2d1kb8p2zv5-3.start {
    counter-reset: lst-ctn-kix_k2d1kb8p2zv5-3 0
}
.lst-kix_ao4h9rgb0ekx-5>li:before {
    content: "\0025a0  "
}
.lst-kix_ao4h9rgb0ekx-6>li:before {
    content: "\0025cf  "
}
ol.lst-kix_lqlac0obey9y-6.start {
    counter-reset: lst-ctn-kix_lqlac0obey9y-6 0
}
.lst-kix_aeeme22hncrc-6>li:before {
    content: "\0025cf  "
}
.lst-kix_aeeme22hncrc-5>li:before {
    content: "\0025a0  "
}
.lst-kix_bde8pay3hzip-5>li:before {
    content: "\0025a0  "
}
.lst-kix_bde8pay3hzip-4>li:before {
    content: "\0025cb  "
}
.lst-kix_9cyha3bjgec2-8>li:before {
    content: "\0025a0  "
}
.lst-kix_z7nrbovt5e2k-2>li:before {
    content: "\0025a0  "
}
.lst-kix_z7nrbovt5e2k-6>li:before {
    content: "\0025cf  "
}
.lst-kix_9cyha3bjgec2-5>li:before {
    content: "\0025a0  "
}
.lst-kix_bde8pay3hzip-8>li:before {
    content: "\0025a0  "
}
.lst-kix_z7nrbovt5e2k-3>li:before {
    content: "\0025cf  "
}
.lst-kix_ao7ckee9pbpp-0>li {
    counter-increment: lst-ctn-kix_ao7ckee9pbpp-0
}
ol.lst-kix_ao7ckee9pbpp-4 {
    list-style-type: none
}
ol.lst-kix_ao7ckee9pbpp-3 {
    list-style-type: none
}
ol.lst-kix_ao7ckee9pbpp-2 {
    list-style-type: none
}
ol.lst-kix_ao7ckee9pbpp-1 {
    list-style-type: none
}
ol.lst-kix_ao7ckee9pbpp-0 {
    list-style-type: none
}
.lst-kix_x0bujimllc2-7>li:before {
    content: "\0025cb  "
}
.lst-kix_x0bujimllc2-0>li:before {
    content: "\0025cf  "
}
ol.lst-kix_ao7ckee9pbpp-8 {
    list-style-type: none
}
ol.lst-kix_ao7ckee9pbpp-7 {
    list-style-type: none
}
.lst-kix_osealum981ss-4>li:before {
    content: "\0025cb  "
}
ol.lst-kix_ao7ckee9pbpp-6 {
    list-style-type: none
}
ol.lst-kix_ao7ckee9pbpp-5 {
    list-style-type: none
}
ol.lst-kix_lqlac0obey9y-6 {
    list-style-type: none
}
.lst-kix_so04ye1qhvjl-6>li:before {
    content: "-  "
}
ol.lst-kix_lqlac0obey9y-5 {
    list-style-type: none
}
ol.lst-kix_lqlac0obey9y-8 {
    list-style-type: none
}
ol.lst-kix_lqlac0obey9y-7 {
    list-style-type: none
}
ol.lst-kix_lqlac0obey9y-2 {
    list-style-type: none
}
ol.lst-kix_lqlac0obey9y-1 {
    list-style-type: none
}
ol.lst-kix_lqlac0obey9y-4 {
    list-style-type: none
}
ol.lst-kix_lqlac0obey9y-3 {
    list-style-type: none
}
ol.lst-kix_k2d1kb8p2zv5-1.start {
    counter-reset: lst-ctn-kix_k2d1kb8p2zv5-1 0
}
ol.lst-kix_lqlac0obey9y-0 {
    list-style-type: none
}
.lst-kix_b4nize190pu8-5>li:before {
    content: "\0025a0  "
}
.lst-kix_osealum981ss-8>li:before {
    content: "\0025a0  "
}
.lst-kix_v6ecskul40c2-5>li:before {
    content: "\0025a0  "
}
.lst-kix_pd9b5gi6xvdb-8>li:before {
    content: "\0025a0  "
}
.lst-kix_ao7ckee9pbpp-4>li:before {
    content: "" counter(lst-ctn-kix_ao7ckee9pbpp-4, lower-latin) ". "
}
.lst-kix_ao7ckee9pbpp-8>li:before {
    content: "" counter(lst-ctn-kix_ao7ckee9pbpp-8, lower-roman) ". "
}
.lst-kix_v6ecskul40c2-1>li:before {
    content: "\0025cb  "
}
.lst-kix_pd9b5gi6xvdb-4>li:before {
    content: "\0025cb  "
}
.lst-kix_b4nize190pu8-1>li:before {
    content: "\0025cb  "
}
.lst-kix_kguy781w2tpm-7>li:before {
    content: "\0025cb  "
}
ul.lst-kix_vnyj1ikev0ct-2 {
    list-style-type: none
}
ul.lst-kix_vnyj1ikev0ct-1 {
    list-style-type: none
}
ul.lst-kix_vnyj1ikev0ct-0 {
    list-style-type: none
}
ul.lst-kix_vnyj1ikev0ct-6 {
    list-style-type: none
}
ul.lst-kix_vnyj1ikev0ct-5 {
    list-style-type: none
}
ul.lst-kix_vnyj1ikev0ct-4 {
    list-style-type: none
}
ul.lst-kix_vnyj1ikev0ct-3 {
    list-style-type: none
}
.lst-kix_kguy781w2tpm-3>li:before {
    content: "\0025cf  "
}
ul.lst-kix_vnyj1ikev0ct-8 {
    list-style-type: none
}
ul.lst-kix_vnyj1ikev0ct-7 {
    list-style-type: none
}
.lst-kix_2qrmnwynf1xh-3>li:before {
    content: "\0025cf  "
}
.lst-kix_xckeupt8b5jl-1>li:before {
    content: "\0025cb  "
}
ul.lst-kix_so04ye1qhvjl-6 {
    list-style-type: none
}
ol.lst-kix_k2d1kb8p2zv5-0.start {
    counter-reset: lst-ctn-kix_k2d1kb8p2zv5-0 0
}
ul.lst-kix_so04ye1qhvjl-5 {
    list-style-type: none
}
ul.lst-kix_so04ye1qhvjl-8 {
    list-style-type: none
}
ul.lst-kix_so04ye1qhvjl-7 {
    list-style-type: none
}
ul.lst-kix_bde8pay3hzip-8 {
    list-style-type: none
}
.lst-kix_pd9b5gi6xvdb-0>li:before {
    content: "\0025cf  "
}
.lst-kix_g7ryix1tcu3y-1>li:before {
    content: "\0025cb  "
}
.lst-kix_ao7ckee9pbpp-0>li:before {
    content: "" counter(lst-ctn-kix_ao7ckee9pbpp-0, decimal) ". "
}
.lst-kix_2qrmnwynf1xh-7>li:before {
    content: "\0025cb  "
}
.lst-kix_k2d1kb8p2zv5-3>li:before {
    content: "" counter(lst-ctn-kix_k2d1kb8p2zv5-3, decimal) ". "
}
.lst-kix_k2d1kb8p2zv5-7>li {
    counter-increment: lst-ctn-kix_k2d1kb8p2zv5-7
}
.lst-kix_g7ryix1tcu3y-5>li:before {
    content: "\0025a0  "
}
.lst-kix_h3upo6j1qfko-1>li:before {
    content: "\0025cb  "
}
.lst-kix_xckeupt8b5jl-5>li:before {
    content: "\0025a0  "
}
.lst-kix_fdpwr6rljdml-3>li:before {
    content: "\0025cf  "
}
.lst-kix_fdpwr6rljdml-7>li:before {
    content: "\0025cb  "
}
ol.lst-kix_lqlac0obey9y-8.start {
    counter-reset: lst-ctn-kix_lqlac0obey9y-8 0
}
ul.lst-kix_9cyha3bjgec2-4 {
    list-style-type: none
}
ul.lst-kix_9cyha3bjgec2-3 {
    list-style-type: none
}
ul.lst-kix_9cyha3bjgec2-2 {
    list-style-type: none
}
.lst-kix_lqlac0obey9y-3>li {
    counter-increment: lst-ctn-kix_lqlac0obey9y-3
}
ul.lst-kix_9cyha3bjgec2-1 {
    list-style-type: none
}
ul.lst-kix_9cyha3bjgec2-8 {
    list-style-type: none
}
ul.lst-kix_9cyha3bjgec2-7 {
    list-style-type: none
}
ul.lst-kix_9cyha3bjgec2-6 {
    list-style-type: none
}
ul.lst-kix_9cyha3bjgec2-5 {
    list-style-type: none
}
ul.lst-kix_bde8pay3hzip-7 {
    list-style-type: none
}
ul.lst-kix_tbjp93w5xcts-1 {
    list-style-type: none
}
ul.lst-kix_bde8pay3hzip-6 {
    list-style-type: none
}
ul.lst-kix_tbjp93w5xcts-2 {
    list-style-type: none
}
ul.lst-kix_bde8pay3hzip-5 {
    list-style-type: none
}
.lst-kix_g7gt31185t1l-1>li:before {
    content: "-  "
}
ul.lst-kix_bde8pay3hzip-4 {
    list-style-type: none
}
ul.lst-kix_tbjp93w5xcts-0 {
    list-style-type: none
}
ul.lst-kix_bde8pay3hzip-3 {
    list-style-type: none
}
ul.lst-kix_bde8pay3hzip-2 {
    list-style-type: none
}
ul.lst-kix_bde8pay3hzip-1 {
    list-style-type: none
}
ul.lst-kix_bde8pay3hzip-0 {
    list-style-type: none
}
ul.lst-kix_tbjp93w5xcts-7 {
    list-style-type: none
}
ul.lst-kix_tbjp93w5xcts-8 {
    list-style-type: none
}
.lst-kix_osealum981ss-0>li:before {
    content: "\0025cf  "
}
ul.lst-kix_tbjp93w5xcts-5 {
    list-style-type: none
}
.lst-kix_ao7ckee9pbpp-5>li {
    counter-increment: lst-ctn-kix_ao7ckee9pbpp-5
}
ul.lst-kix_tbjp93w5xcts-6 {
    list-style-type: none
}
ul.lst-kix_tbjp93w5xcts-3 {
    list-style-type: none
}
ul.lst-kix_tbjp93w5xcts-4 {
    list-style-type: none
}
ul.lst-kix_rhatjkb0huf0-0 {
    list-style-type: none
}
ul.lst-kix_rhatjkb0huf0-1 {
    list-style-type: none
}
.lst-kix_tbjp93w5xcts-2>li:before {
    content: "\0025a0  "
}
ul.lst-kix_x0bujimllc2-3 {
    list-style-type: none
}
ul.lst-kix_x0bujimllc2-4 {
    list-style-type: none
}
ul.lst-kix_x0bujimllc2-5 {
    list-style-type: none
}
ul.lst-kix_x0bujimllc2-6 {
    list-style-type: none
}
.lst-kix_tbjp93w5xcts-1>li:before {
    content: "\0025cb  "
}
ul.lst-kix_x0bujimllc2-0 {
    list-style-type: none
}
ul.lst-kix_x0bujimllc2-1 {
    list-style-type: none
}
ul.lst-kix_x0bujimllc2-2 {
    list-style-type: none
}
.lst-kix_tbjp93w5xcts-0>li:before {
    content: "\0025cf  "
}
.lst-kix_tbjp93w5xcts-5>li:before {
    content: "\0025a0  "
}
.lst-kix_tbjp93w5xcts-7>li:before {
    content: "\0025cb  "
}
ul.lst-kix_9cyha3bjgec2-0 {
    list-style-type: none
}
.lst-kix_tbjp93w5xcts-6>li:before {
    content: "\0025cf  "
}
ul.lst-kix_rhatjkb0huf0-2 {
    list-style-type: none
}
ul.lst-kix_rhatjkb0huf0-3 {
    list-style-type: none
}
ul.lst-kix_rhatjkb0huf0-4 {
    list-style-type: none
}
ul.lst-kix_rhatjkb0huf0-5 {
    list-style-type: none
}
.lst-kix_tbjp93w5xcts-3>li:before {
    content: "\0025cf  "
}
ul.lst-kix_rhatjkb0huf0-6 {
    list-style-type: none
}
ul.lst-kix_rhatjkb0huf0-7 {
    list-style-type: none
}
ul.lst-kix_rhatjkb0huf0-8 {
    list-style-type: none
}
.lst-kix_tbjp93w5xcts-4>li:before {
    content: "\0025cb  "
}
.lst-kix_55qcczpq6rq9-2>li:before {
    content: "\0025a0  "
}
.lst-kix_ao7ckee9pbpp-4>li {
    counter-increment: lst-ctn-kix_ao7ckee9pbpp-4
}
.lst-kix_55qcczpq6rq9-3>li:before {
    content: "\0025cf  "
}
.lst-kix_55qcczpq6rq9-4>li:before {
    content: "\0025cb  "
}
.lst-kix_k2d1kb8p2zv5-0>li {
    counter-increment: lst-ctn-kix_k2d1kb8p2zv5-0
}
.lst-kix_tbjp93w5xcts-8>li:before {
    content: "\0025a0  "
}
ul.lst-kix_g7ryix1tcu3y-0 {
    list-style-type: none
}
ul.lst-kix_g7ryix1tcu3y-1 {
    list-style-type: none
}
ol.lst-kix_ao7ckee9pbpp-0.start {
    counter-reset: lst-ctn-kix_ao7ckee9pbpp-0 0
}
ul.lst-kix_g7ryix1tcu3y-2 {
    list-style-type: none
}
.lst-kix_55qcczpq6rq9-5>li:before {
    content: "\0025a0  "
}
.lst-kix_55qcczpq6rq9-7>li:before {
    content: "\0025cb  "
}
ul.lst-kix_g7ryix1tcu3y-3 {
    list-style-type: none
}
ol.lst-kix_lqlac0obey9y-4.start {
    counter-reset: lst-ctn-kix_lqlac0obey9y-4 0
}
ul.lst-kix_g7ryix1tcu3y-4 {
    list-style-type: none
}
ul.lst-kix_g7ryix1tcu3y-5 {
    list-style-type: none
}
ul.lst-kix_g7ryix1tcu3y-6 {
    list-style-type: none
}
.lst-kix_55qcczpq6rq9-6>li:before {
    content: "\0025cf  "
}
ul.lst-kix_g7ryix1tcu3y-7 {
    list-style-type: none
}
ul.lst-kix_g7ryix1tcu3y-8 {
    list-style-type: none
}
.lst-kix_55qcczpq6rq9-8>li:before {
    content: "\0025a0  "
}
.lst-kix_vnyj1ikev0ct-3>li:before {
    content: "\0025cf  "
}
.lst-kix_vnyj1ikev0ct-1>li:before {
    content: "\0025cb  "
}
.lst-kix_vnyj1ikev0ct-5>li:before {
    content: "\0025a0  "
}
.lst-kix_vnyj1ikev0ct-0>li:before {
    content: "\0025cf  "
}
.lst-kix_vnyj1ikev0ct-4>li:before {
    content: "\0025cb  "
}
.lst-kix_vnyj1ikev0ct-2>li:before {
    content: "\0025a0  "
}
ol.lst-kix_ao7ckee9pbpp-7.start {
    counter-reset: lst-ctn-kix_ao7ckee9pbpp-7 0
}
.lst-kix_egnx6h47ljtr-2>li:before {
    content: "\0025a0  "
}
.lst-kix_egnx6h47ljtr-0>li:before {
    content: "\0025cf  "
}
.lst-kix_egnx6h47ljtr-1>li:before {
    content: "\0025cb  "
}
.lst-kix_v6ecskul40c2-0>li:before {
    content: "\0025cf  "
}
.lst-kix_55qcczpq6rq9-1>li:before {
    content: "\0025cb  "
}
.lst-kix_55qcczpq6rq9-0>li:before {
    content: "\0025cf  "
}
ul.lst-kix_g7gt31185t1l-0 {
    list-style-type: none
}
ul.lst-kix_g7gt31185t1l-3 {
    list-style-type: none
}
.lst-kix_egnx6h47ljtr-8>li:before {
    content: "\0025a0  "
}
ul.lst-kix_g7gt31185t1l-4 {
    list-style-type: none
}
ul.lst-kix_g7gt31185t1l-1 {
    list-style-type: none
}
ul.lst-kix_g7gt31185t1l-2 {
    list-style-type: none
}
.lst-kix_so04ye1qhvjl-4>li:before {
    content: "-  "
}
ol.lst-kix_k2d1kb8p2zv5-4.start {
    counter-reset: lst-ctn-kix_k2d1kb8p2zv5-4 0
}
.lst-kix_egnx6h47ljtr-3>li:before {
    content: "\0025cf  "
}
ul.lst-kix_x0bujimllc2-7 {
    list-style-type: none
}
.lst-kix_egnx6h47ljtr-4>li:before {
    content: "\0025cb  "
}
.lst-kix_egnx6h47ljtr-5>li:before {
    content: "\0025a0  "
}
.lst-kix_so04ye1qhvjl-3>li:before {
    content: "-  "
}
ul.lst-kix_x0bujimllc2-8 {
    list-style-type: none
}
.lst-kix_vnyj1ikev0ct-8>li:before {
    content: "\0025a0  "
}
.lst-kix_vnyj1ikev0ct-7>li:before {
    content: "\0025cb  "
}
.lst-kix_egnx6h47ljtr-6>li:before {
    content: "\0025cf  "
}
.lst-kix_egnx6h47ljtr-7>li:before {
    content: "\0025cb  "
}
.lst-kix_so04ye1qhvjl-1>li:before {
    content: "-  "
}
.lst-kix_so04ye1qhvjl-2>li:before {
    content: "-  "
}
.lst-kix_so04ye1qhvjl-0>li:before {
    content: "-  "
}
.lst-kix_vnyj1ikev0ct-6>li:before {
    content: "\0025cf  "
}
ul.lst-kix_xckeupt8b5jl-3 {
    list-style-type: none
}
ul.lst-kix_xckeupt8b5jl-2 {
    list-style-type: none
}
ul.lst-kix_xckeupt8b5jl-1 {
    list-style-type: none
}
ul.lst-kix_xckeupt8b5jl-0 {
    list-style-type: none
}
ul.lst-kix_xckeupt8b5jl-7 {
    list-style-type: none
}
ul.lst-kix_xckeupt8b5jl-6 {
    list-style-type: none
}
ul.lst-kix_xckeupt8b5jl-5 {
    list-style-type: none
}
ul.lst-kix_xckeupt8b5jl-4 {
    list-style-type: none
}
ol.lst-kix_k2d1kb8p2zv5-1 {
    list-style-type: none
}
.lst-kix_so04ye1qhvjl-5>li:before {
    content: "-  "
}
ol.lst-kix_k2d1kb8p2zv5-2 {
    list-style-type: none
}
ol.lst-kix_k2d1kb8p2zv5-3 {
    list-style-type: none
}
ol.lst-kix_k2d1kb8p2zv5-4 {
    list-style-type: none
}
.lst-kix_so04ye1qhvjl-7>li:before {
    content: "-  "
}
.lst-kix_b4nize190pu8-4>li:before {
    content: "\0025cb  "
}
.lst-kix_b4nize190pu8-8>li:before {
    content: "\0025a0  "
}
ol.lst-kix_k2d1kb8p2zv5-0 {
    list-style-type: none
}
.lst-kix_k2d1kb8p2zv5-4>li {
    counter-increment: lst-ctn-kix_k2d1kb8p2zv5-4
}
ul.lst-kix_g7gt31185t1l-7 {
    list-style-type: none
}
ul.lst-kix_g7gt31185t1l-8 {
    list-style-type: none
}
ul.lst-kix_g7gt31185t1l-5 {
    list-style-type: none
}
ul.lst-kix_g7gt31185t1l-6 {
    list-style-type: none
}
ol.lst-kix_k2d1kb8p2zv5-5 {
    list-style-type: none
}
ol.lst-kix_k2d1kb8p2zv5-6 {
    list-style-type: none
}
.lst-kix_b4nize190pu8-6>li:before {
    content: "\0025cf  "
}
ol.lst-kix_k2d1kb8p2zv5-7 {
    list-style-type: none
}
ol.lst-kix_k2d1kb8p2zv5-8 {
    list-style-type: none
}
.lst-kix_pd9b5gi6xvdb-7>li:before {
    content: "\0025cb  "
}
.lst-kix_pd9b5gi6xvdb-5>li:before {
    content: "\0025a0  "
}
.lst-kix_v6ecskul40c2-4>li:before {
    content: "\0025cb  "
}
.lst-kix_v6ecskul40c2-8>li:before {
    content: "\0025a0  "
}
.lst-kix_b4nize190pu8-0>li:before {
    content: "\0025cf  "
}
.lst-kix_pd9b5gi6xvdb-3>li:before {
    content: "\0025cf  "
}
.lst-kix_ao7ckee9pbpp-1>li {
    counter-increment: lst-ctn-kix_ao7ckee9pbpp-1
}
.lst-kix_kguy781w2tpm-8>li:before {
    content: "\0025a0  "
}
.lst-kix_v6ecskul40c2-2>li:before {
    content: "\0025a0  "
}
.lst-kix_b4nize190pu8-2>li:before {
    content: "\0025a0  "
}
.lst-kix_kguy781w2tpm-4>li:before {
    content: "\0025cb  "
}
.lst-kix_kguy781w2tpm-6>li:before {
    content: "\0025cf  "
}
.lst-kix_k2d1kb8p2zv5-3>li {
    counter-increment: lst-ctn-kix_k2d1kb8p2zv5-3
}
.lst-kix_kguy781w2tpm-2>li:before {
    content: "\0025a0  "
}
ul.lst-kix_xckeupt8b5jl-8 {
    list-style-type: none
}
.lst-kix_v6ecskul40c2-6>li:before {
    content: "\0025cf  "
}
ol.lst-kix_lqlac0obey9y-2.start {
    counter-reset: lst-ctn-kix_lqlac0obey9y-2 0
}
.lst-kix_kguy781w2tpm-0>li:before {
    content: "\0025cf  "
}
.lst-kix_pd9b5gi6xvdb-1>li:before {
    content: "\0025cb  "
}
ol.lst-kix_k2d1kb8p2zv5-6.start {
    counter-reset: lst-ctn-kix_k2d1kb8p2zv5-6 0
}
.lst-kix_fdpwr6rljdml-6>li:before {
    content: "\0025cf  "
}
.lst-kix_lqlac0obey9y-6>li {
    counter-increment: lst-ctn-kix_lqlac0obey9y-6
}
.lst-kix_fdpwr6rljdml-2>li:before {
    content: "\0025a0  "
}
.lst-kix_fdpwr6rljdml-0>li:before {
    content: "\0025cf  "
}
.lst-kix_fdpwr6rljdml-8>li:before {
    content: "\0025a0  "
}
.lst-kix_ao7ckee9pbpp-8>li {
    counter-increment: lst-ctn-kix_ao7ckee9pbpp-8
}
.lst-kix_fdpwr6rljdml-4>li:before {
    content: "\0025cb  "
}
.lst-kix_or8pcwk62bfz-7>li:before {
    content: "-  "
}
.lst-kix_or8pcwk62bfz-6>li:before {
    content: "-  "
}
.lst-kix_z7nrbovt5e2k-8>li:before {
    content: "\0025a0  "
}
.lst-kix_x7qfbxbrd6e4-4>li:before {
    content: "-  "
}
ul.lst-kix_osealum981ss-1 {
    list-style-type: none
}
ul.lst-kix_osealum981ss-2 {
    list-style-type: none
}
ul.lst-kix_osealum981ss-0 {
    list-style-type: none
}
.lst-kix_bde8pay3hzip-2>li:before {
    content: "\0025a0  "
}
ul.lst-kix_osealum981ss-5 {
    list-style-type: none
}
ul.lst-kix_kguy781w2tpm-7 {
    list-style-type: none
}
ul.lst-kix_osealum981ss-6 {
    list-style-type: none
}
ul.lst-kix_kguy781w2tpm-8 {
    list-style-type: none
}
ol.lst-kix_k2d1kb8p2zv5-8.start {
    counter-reset: lst-ctn-kix_k2d1kb8p2zv5-8 0
}
.lst-kix_lqlac0obey9y-7>li {
    counter-increment: lst-ctn-kix_lqlac0obey9y-7
}
.lst-kix_bde8pay3hzip-3>li:before {
    content: "\0025cf  "
}
ul.lst-kix_osealum981ss-3 {
    list-style-type: none
}
ul.lst-kix_osealum981ss-4 {
    list-style-type: none
}
ul.lst-kix_kguy781w2tpm-3 {
    list-style-type: none
}
.lst-kix_or8pcwk62bfz-2>li:before {
    content: "-  "
}
ul.lst-kix_kguy781w2tpm-4 {
    list-style-type: none
}
.lst-kix_x7qfbxbrd6e4-5>li:before {
    content: "-  "
}
ul.lst-kix_osealum981ss-7 {
    list-style-type: none
}
ul.lst-kix_kguy781w2tpm-5 {
    list-style-type: none
}
ul.lst-kix_osealum981ss-8 {
    list-style-type: none
}
ul.lst-kix_kguy781w2tpm-6 {
    list-style-type: none
}
.lst-kix_rhatjkb0huf0-3>li:before {
    content: "\0025cf  "
}
.lst-kix_or8pcwk62bfz-3>li:before {
    content: "-  "
}
ul.lst-kix_kguy781w2tpm-0 {
    list-style-type: none
}
ul.lst-kix_kguy781w2tpm-1 {
    list-style-type: none
}
ul.lst-kix_kguy781w2tpm-2 {
    list-style-type: none
}
.lst-kix_x7qfbxbrd6e4-8>li:before {
    content: "-  "
}
.lst-kix_rhatjkb0huf0-4>li:before {
    content: "\0025cb  "
}
.lst-kix_rhatjkb0huf0-7>li:before {
    content: "\0025cb  "
}
ol.lst-kix_ao7ckee9pbpp-3.start {
    counter-reset: lst-ctn-kix_ao7ckee9pbpp-3 0
}
.lst-kix_secj84g03ko5-0>li:before {
    content: "-  "
}
ol.lst-kix_lqlac0obey9y-1.start {
    counter-reset: lst-ctn-kix_lqlac0obey9y-1 0
}
.lst-kix_h3upo6j1qfko-3>li:before {
    content: "\0025cf  "
}
.lst-kix_secj84g03ko5-4>li:before {
    content: "-  "
}
.lst-kix_h3upo6j1qfko-6>li:before {
    content: "\0025cf  "
}
.lst-kix_rhatjkb0huf0-8>li:before {
    content: "\0025a0  "
}
.lst-kix_h3upo6j1qfko-7>li:before {
    content: "\0025cb  "
}
.lst-kix_x7qfbxbrd6e4-1>li:before {
    content: "-  "
}
.lst-kix_secj84g03ko5-3>li:before {
    content: "-  "
}
.lst-kix_x7qfbxbrd6e4-0>li:before {
    content: "-  "
}
.lst-kix_aeeme22hncrc-0>li:before {
    content: "\0025cf  "
}
.lst-kix_k2d1kb8p2zv5-8>li {
    counter-increment: lst-ctn-kix_k2d1kb8p2zv5-8
}
.lst-kix_ao4h9rgb0ekx-0>li:before {
    content: "\0025cf  "
}
ul.lst-kix_h3upo6j1qfko-3 {
    list-style-type: none
}
.lst-kix_9cyha3bjgec2-2>li:before {
    content: "\0025a0  "
}
ul.lst-kix_h3upo6j1qfko-2 {
    list-style-type: none
}
.lst-kix_aeeme22hncrc-3>li:before {
    content: "\0025cf  "
}
ul.lst-kix_h3upo6j1qfko-1 {
    list-style-type: none
}
ul.lst-kix_h3upo6j1qfko-0 {
    list-style-type: none
}
.lst-kix_9cyha3bjgec2-3>li:before {
    content: "\0025cf  "
}
.lst-kix_aeeme22hncrc-8>li:before {
    content: "\0025a0  "
}
.lst-kix_ao4h9rgb0ekx-7>li:before {
    content: "\0025cb  "
}
.lst-kix_secj84g03ko5-7>li:before {
    content: "-  "
}
.lst-kix_secj84g03ko5-8>li:before {
    content: "-  "
}
.lst-kix_aeeme22hncrc-7>li:before {
    content: "\0025cb  "
}
.lst-kix_aeeme22hncrc-4>li:before {
    content: "\0025cb  "
}
ul.lst-kix_h3upo6j1qfko-7 {
    list-style-type: none
}
ul.lst-kix_h3upo6j1qfko-6 {
    list-style-type: none
}
.lst-kix_ao4h9rgb0ekx-3>li:before {
    content: "\0025cf  "
}
.lst-kix_ao4h9rgb0ekx-4>li:before {
    content: "\0025cb  "
}
ul.lst-kix_h3upo6j1qfko-5 {
    list-style-type: none
}
ul.lst-kix_h3upo6j1qfko-4 {
    list-style-type: none
}
ul.lst-kix_h3upo6j1qfko-8 {
    list-style-type: none
}
.lst-kix_z7nrbovt5e2k-0>li:before {
    content: "\0025cf  "
}
ol.lst-kix_ao7ckee9pbpp-8.start {
    counter-reset: lst-ctn-kix_ao7ckee9pbpp-8 0
}
.lst-kix_bde8pay3hzip-6>li:before {
    content: "\0025cf  "
}
.lst-kix_z7nrbovt5e2k-4>li:before {
    content: "\0025cb  "
}
.lst-kix_9cyha3bjgec2-7>li:before {
    content: "\0025cb  "
}
.lst-kix_9cyha3bjgec2-6>li:before {
    content: "\0025cf  "
}
.lst-kix_z7nrbovt5e2k-1>li:before {
    content: "\0025cb  "
}
.lst-kix_z7nrbovt5e2k-5>li:before {
    content: "\0025a0  "
}
.lst-kix_bde8pay3hzip-7>li:before {
    content: "\0025cb  "
}
.lst-kix_x0bujimllc2-2>li:before {
    content: "\0025a0  "
}
.lst-kix_x0bujimllc2-5>li:before {
    content: "\0025a0  "
}
.lst-kix_lqlac0obey9y-2>li {
    counter-increment: lst-ctn-kix_lqlac0obey9y-2
}
ol.lst-kix_ao7ckee9pbpp-2.start {
    counter-reset: lst-ctn-kix_ao7ckee9pbpp-2 0
}
ul.lst-kix_fdpwr6rljdml-2 {
    list-style-type: none
}
ul.lst-kix_fdpwr6rljdml-1 {
    list-style-type: none
}
.lst-kix_osealum981ss-2>li:before {
    content: "\0025a0  "
}
ul.lst-kix_fdpwr6rljdml-4 {
    list-style-type: none
}
ul.lst-kix_fdpwr6rljdml-3 {
    list-style-type: none
}
ul.lst-kix_fdpwr6rljdml-0 {
    list-style-type: none
}
.lst-kix_b4nize190pu8-7>li:before {
    content: "\0025cb  "
}
.lst-kix_osealum981ss-6>li:before {
    content: "\0025cf  "
}
.lst-kix_so04ye1qhvjl-8>li:before {
    content: "-  "
}
ul.lst-kix_2qrmnwynf1xh-4 {
    list-style-type: none
}
ul.lst-kix_2qrmnwynf1xh-5 {
    list-style-type: none
}
ul.lst-kix_2qrmnwynf1xh-2 {
    list-style-type: none
}
ul.lst-kix_2qrmnwynf1xh-3 {
    list-style-type: none
}
ul.lst-kix_2qrmnwynf1xh-0 {
    list-style-type: none
}
ul.lst-kix_2qrmnwynf1xh-1 {
    list-style-type: none
}
.lst-kix_pd9b5gi6xvdb-6>li:before {
    content: "\0025cf  "
}
.lst-kix_b4nize190pu8-3>li:before {
    content: "\0025cf  "
}
ul.lst-kix_2qrmnwynf1xh-8 {
    list-style-type: none
}
.lst-kix_v6ecskul40c2-3>li:before {
    content: "\0025cf  "
}
ul.lst-kix_2qrmnwynf1xh-6 {
    list-style-type: none
}
.lst-kix_ao7ckee9pbpp-2>li:before {
    content: "" counter(lst-ctn-kix_ao7ckee9pbpp-2, lower-roman) ". "
}
ul.lst-kix_2qrmnwynf1xh-7 {
    list-style-type: none
}
.lst-kix_kguy781w2tpm-5>li:before {
    content: "\0025a0  "
}
.lst-kix_k2d1kb8p2zv5-6>li {
    counter-increment: lst-ctn-kix_k2d1kb8p2zv5-6
}
.lst-kix_v6ecskul40c2-7>li:before {
    content: "\0025cb  "
}
.lst-kix_ao7ckee9pbpp-6>li:before {
    content: "" counter(lst-ctn-kix_ao7ckee9pbpp-6, decimal) ". "
}
.lst-kix_2qrmnwynf1xh-1>li:before {
    content: "\0025cb  "
}
.lst-kix_2qrmnwynf1xh-5>li:before {
    content: "\0025a0  "
}
ol.lst-kix_ao7ckee9pbpp-4.start {
    counter-reset: lst-ctn-kix_ao7ckee9pbpp-4 0
}
.lst-kix_xckeupt8b5jl-3>li:before {
    content: "\0025cf  "
}
.lst-kix_xckeupt8b5jl-7>li:before {
    content: "\0025cb  "
}
.lst-kix_kguy781w2tpm-1>li:before {
    content: "\0025cb  "
}
.lst-kix_g7ryix1tcu3y-7>li:before {
    content: "\0025cb  "
}
.lst-kix_g7ryix1tcu3y-3>li:before {
    content: "\0025cf  "
}
.lst-kix_k2d1kb8p2zv5-5>li:before {
    content: "" counter(lst-ctn-kix_k2d1kb8p2zv5-5, lower-roman) ". "
}
.lst-kix_pd9b5gi6xvdb-2>li:before {
    content: "\0025a0  "
}
.lst-kix_k2d1kb8p2zv5-1>li {
    counter-increment: lst-ctn-kix_k2d1kb8p2zv5-1
}
.lst-kix_fdpwr6rljdml-5>li:before {
    content: "\0025a0  "
}
ul.lst-kix_b4nize190pu8-6 {
    list-style-type: none
}
.lst-kix_k2d1kb8p2zv5-1>li:before {
    content: "" counter(lst-ctn-kix_k2d1kb8p2zv5-1, lower-latin) ". "
}
ul.lst-kix_b4nize190pu8-7 {
    list-style-type: none
}
ul.lst-kix_b4nize190pu8-8 {
    list-style-type: none
}
ul.lst-kix_b4nize190pu8-2 {
    list-style-type: none
}
ul.lst-kix_b4nize190pu8-3 {
    list-style-type: none
}
.lst-kix_fdpwr6rljdml-1>li:before {
    content: "\0025cb  "
}
ul.lst-kix_b4nize190pu8-4 {
    list-style-type: none
}
ul.lst-kix_b4nize190pu8-5 {
    list-style-type: none
}
ul.lst-kix_b4nize190pu8-0 {
    list-style-type: none
}
.lst-kix_rhatjkb0huf0-0>li:before {
    content: "\0025cf  "
}
ul.lst-kix_b4nize190pu8-1 {
    list-style-type: none
}
ul.lst-kix_fdpwr6rljdml-6 {
    list-style-type: none
}
ul.lst-kix_fdpwr6rljdml-5 {
    list-style-type: none
}
ul.lst-kix_fdpwr6rljdml-8 {
    list-style-type: none
}
ul.lst-kix_fdpwr6rljdml-7 {
    list-style-type: none
}
ol.lst-kix_ao7ckee9pbpp-5.start {
    counter-reset: lst-ctn-kix_ao7ckee9pbpp-5 0
}
.lst-kix_lqlac0obey9y-0>li:before {
    content: "" counter(lst-ctn-kix_lqlac0obey9y-0, decimal) ". "
}
ol {
    margin: 0;
    padding: 0
}
.fullscreen {
    position: absolute;
    padding-right: 4px;
    background-color: white;
    top: -12px;
    right: 5px;
    padding-left: 3px;
    border-top: 1px solid black;
    border-left: 1px solid black;
    border-right: 1px solid black;
    font-size: 9pt;
    font-family: "Arial";
    font-style: normal
}
.c1 {
    color: #000000;
    font-weight: 400;
    text-decoration: none;
    vertical-align: baseline;
    font-size: 11pt;
    font-family: "Arial";
    font-style: normal
}
.c2 {
    color: #000000;
    font-weight: 400;
    text-decoration: none;
    vertical-align: baseline;
    font-size: 9pt;
    font-family: "Arial";
    font-style: normal
}
.c21 {
    padding-top: 0pt;
    padding-bottom: 0pt;
    orphans: 2;
    widows: 2;
    text-align: center
}
.c28 {
    margin-left: 22pt;
    padding-top: 11pt;
    text-indent: -22pt;
    padding-bottom: 0pt;
    text-align: left
}
.c16 {
    margin-left: 22pt;
    padding-top: 0pt;
    text-indent: -22pt;
    padding-bottom: 11pt;
    text-align: left
}
.c17 {
    padding-top: 0pt;
    padding-bottom: 0pt;
    orphans: 2;
    widows: 2;
    text-align: left
}
.c6 {
    padding-top: 0pt;
    padding-bottom: 0pt;
    orphans: 2;
    widows: 2;
    text-align: justify
}
.c32 {
    margin-left: 22pt;
    padding-top: 0pt;
    text-indent: -22pt;
    padding-bottom: 11pt;
    text-align: left
}
.c15 {
    margin-left: 22pt;
    padding-top: 0pt;
    text-indent: -22pt;
    padding-bottom: 0pt;
    text-align: left
}
.c7 {
    font-size: 11pt;
    font-style: normal;
    color: #000000;
    font-weight: 700;
    text-decoration: none
}
.c14 {
    padding-top: 20pt;
    padding-bottom: 6pt;
    page-break-after: avoid;
    text-align: left
}
.c20 {
    padding-top: 20pt;
    padding-bottom: 6pt;
    page-break-after: avoid;
    text-align: justify
}
.c18 {
    color: #000000;
    font-weight: 700;
    text-decoration: none;
    vertical-align: baseline;
    font-family: "Arial"
}
.c31 {
    padding-top: 2pt;
    padding-bottom: 4pt;
    page-break-after: avoid;
    text-align: left
}
.c22 {
    padding-top: 16pt;
    padding-bottom: 4pt;
    page-break-after: avoid;
    text-align: justify
}
.c27 {
    vertical-align: baseline;
    font-size: 14pt;
    font-family: "Arial";
    font-style: normal
}
.c13 {
    vertical-align: baseline;
    font-size: 20pt;
    font-family: "Arial";
    font-style: normal
}
.c5 {
    color: #000000;
    font-weight: 400;
    text-decoration: none
}
.c24 {
    background-color: #ffffff;
    max-width: 668pt;
    padding: 10pt 72pt 72pt 72pt
}
.c0 {
    color: inherit;
    text-decoration: inherit
}
.c8 {
    font-size: 11pt;
    font-style: normal
}
.c29 {
    vertical-align: baseline;
    font-family: "Arial"
}
.c12 {
    color: #000000;
    font-weight: 400
}
.c23 {
    font-weight: 700
}
.c9 {
    height: 11pt
}
.c3 {
    font-size: 9pt
}
.c4 {
    vertical-align: top;
    position: relative;
    top: -0.2em;
    font-size: 8pt;
    padding-left: 2px;
}
.c26 {
    height: 20pt
}
.c10 {
    font-style: italic
}
.c30 {
    color: #000000
}
.c25 {
    color: #1155cc
}
.c19 {
    text-decoration: underline
}
.c11 {
    font-size: 11pt
}
.title {
    padding-top: 0pt;
    color: #000000;
    font-size: 26pt;
    padding-bottom: 3pt;
    font-family: "Arial";
    page-break-after: avoid;
    orphans: 2;
    widows: 2;
    text-align: left
}
.subtitle {
    padding-top: 0pt;
    color: #666666;
    font-size: 15pt;
    padding-bottom: 16pt;
    font-family: "Arial";
    page-break-after: avoid;
    orphans: 2;
    widows: 2;
    text-align: left
}
li {
    color: #000000;
    font-size: 11pt;
    font-family: "Arial"
}
p {
    margin: 0;
    color: #000000;
    font-size: 11pt;
    font-family: "Arial"
}
h1 {
    padding-top: 20pt;
    color: #000000;
    font-size: 20pt;
    padding-bottom: 6pt;
    font-family: "Arial";
    page-break-after: avoid;
    orphans: 2;
    widows: 2;
    text-align: left
}
h2 {
    padding-top: 18pt;
    color: #000000;
    font-size: 16pt;
    padding-bottom: 6pt;
    font-family: "Arial";
    page-break-after: avoid;
    orphans: 2;
    widows: 2;
    text-align: left
}
h3 {
    padding-top: 16pt;
    color: #434343;
    font-size: 14pt;
    padding-bottom: 4pt;
    font-family: "Arial";
    page-break-after: avoid;
    orphans: 2;
    widows: 2;
    text-align: left
}
h4 {
    padding-top: 14pt;
    color: #666666;
    font-size: 12pt;
    padding-bottom: 4pt;
    font-family: "Arial";
    page-break-after: avoid;
    orphans: 2;
    widows: 2;
    text-align: left
}
h5 {
    padding-top: 12pt;
    color: #666666;
    font-size: 11pt;
    padding-bottom: 4pt;
    font-family: "Arial";
    page-break-after: avoid;
    orphans: 2;
    widows: 2;
    text-align: left
}
h6 {
    padding-top: 12pt;
    color: #666666;
    font-size: 11pt;
    padding-bottom: 4pt;
    font-family: "Arial";
    page-break-after: avoid;
    font-style: italic;
    orphans: 2;
    widows: 2;
    text-align: left
}


ol.lst-kix_wu62sfo7w397-8.start {
    counter-reset: lst-ctn-kix_wu62sfo7w397-8 0
}
ol.lst-kix_wu62sfo7w397-5.start {
    counter-reset: lst-ctn-kix_wu62sfo7w397-5 0
}
.lst-kix_k47f4yk9x4ic-5>li:before {
    content: "" counter(lst-ctn-kix_k47f4yk9x4ic-5, lower-roman) ". "
}
.lst-kix_k47f4yk9x4ic-6>li {
    counter-increment: lst-ctn-kix_k47f4yk9x4ic-6
}
.lst-kix_k47f4yk9x4ic-3>li:before {
    content: "" counter(lst-ctn-kix_k47f4yk9x4ic-3, decimal) ". "
}
.lst-kix_k47f4yk9x4ic-7>li:before {
    content: "" counter(lst-ctn-kix_k47f4yk9x4ic-7, lower-latin) ". "
}
.lst-kix_k47f4yk9x4ic-4>li:before {
    content: "" counter(lst-ctn-kix_k47f4yk9x4ic-4, lower-latin) ". "
}
.lst-kix_k47f4yk9x4ic-8>li:before {
    content: "" counter(lst-ctn-kix_k47f4yk9x4ic-8, lower-roman) ". "
}
ol.lst-kix_k47f4yk9x4ic-2.start {
    counter-reset: lst-ctn-kix_k47f4yk9x4ic-2 0
}
.lst-kix_k47f4yk9x4ic-6>li:before {
    content: "" counter(lst-ctn-kix_k47f4yk9x4ic-6, decimal) ". "
}
ol.lst-kix_wu62sfo7w397-2.start {
    counter-reset: lst-ctn-kix_wu62sfo7w397-2 0
}
.lst-kix_k47f4yk9x4ic-0>li:before {
    content: "" counter(lst-ctn-kix_k47f4yk9x4ic-0, decimal) ". "
}
.lst-kix_k47f4yk9x4ic-1>li:before {
    content: "" counter(lst-ctn-kix_k47f4yk9x4ic-1, lower-latin) ". "
}
.lst-kix_k47f4yk9x4ic-2>li:before {
    content: "" counter(lst-ctn-kix_k47f4yk9x4ic-2, lower-roman) ". "
}
.lst-kix_k47f4yk9x4ic-8>li {
    counter-increment: lst-ctn-kix_k47f4yk9x4ic-8
}
.lst-kix_k47f4yk9x4ic-5>li {
    counter-increment: lst-ctn-kix_k47f4yk9x4ic-5
}
ol.lst-kix_k47f4yk9x4ic-4.start {
    counter-reset: lst-ctn-kix_k47f4yk9x4ic-4 0
}
.lst-kix_wu62sfo7w397-8>li:before {
    content: "" counter(lst-ctn-kix_wu62sfo7w397-8, lower-roman) ". "
}
.lst-kix_wu62sfo7w397-7>li:before {
    content: "" counter(lst-ctn-kix_wu62sfo7w397-7, lower-latin) ". "
}
.lst-kix_5fbgyloudqbx-0>li:before {
    content: "\0025cf  "
}
.lst-kix_k47f4yk9x4ic-3>li {
    counter-increment: lst-ctn-kix_k47f4yk9x4ic-3
}
.lst-kix_5fbgyloudqbx-1>li:before {
    content: "\0025cb  "
}
.lst-kix_5fbgyloudqbx-2>li:before {
    content: "\0025a0  "
}
ol.lst-kix_wu62sfo7w397-8 {
    list-style-type: none
}
ol.lst-kix_k47f4yk9x4ic-5.start {
    counter-reset: lst-ctn-kix_k47f4yk9x4ic-5 0
}
.lst-kix_wu62sfo7w397-0>li {
    counter-increment: lst-ctn-kix_wu62sfo7w397-0
}
ol.lst-kix_wu62sfo7w397-2 {
    list-style-type: none
}
ol.lst-kix_wu62sfo7w397-3 {
    list-style-type: none
}
ol.lst-kix_wu62sfo7w397-0 {
    list-style-type: none
}
ol.lst-kix_wu62sfo7w397-1 {
    list-style-type: none
}
ol.lst-kix_wu62sfo7w397-6 {
    list-style-type: none
}
ol.lst-kix_wu62sfo7w397-3.start {
    counter-reset: lst-ctn-kix_wu62sfo7w397-3 0
}
.lst-kix_wu62sfo7w397-3>li {
    counter-increment: lst-ctn-kix_wu62sfo7w397-3
}
.lst-kix_wu62sfo7w397-6>li {
    counter-increment: lst-ctn-kix_wu62sfo7w397-6
}
ol.lst-kix_wu62sfo7w397-7 {
    list-style-type: none
}
ol.lst-kix_wu62sfo7w397-4 {
    list-style-type: none
}
ol.lst-kix_wu62sfo7w397-5 {
    list-style-type: none
}
.lst-kix_wu62sfo7w397-5>li {
    counter-increment: lst-ctn-kix_wu62sfo7w397-5
}
ol.lst-kix_wu62sfo7w397-1.start {
    counter-reset: lst-ctn-kix_wu62sfo7w397-1 0
}
.lst-kix_k47f4yk9x4ic-0>li {
    counter-increment: lst-ctn-kix_k47f4yk9x4ic-0
}
.lst-kix_5fbgyloudqbx-5>li:before {
    content: "\0025a0  "
}
.lst-kix_5fbgyloudqbx-6>li:before {
    content: "\0025cf  "
}
.lst-kix_5fbgyloudqbx-3>li:before {
    content: "\0025cf  "
}
.lst-kix_5fbgyloudqbx-4>li:before {
    content: "\0025cb  "
}
.lst-kix_5fbgyloudqbx-7>li:before {
    content: "\0025cb  "
}
.lst-kix_5fbgyloudqbx-8>li:before {
    content: "\0025a0  "
}
.lst-kix_rrwtpabby6sq-6>li:before {
    content: "\0025cf  "
}
ol.lst-kix_wu62sfo7w397-4.start {
    counter-reset: lst-ctn-kix_wu62sfo7w397-4 0
}
.lst-kix_rrwtpabby6sq-5>li:before {
    content: "\0025a0  "
}
.lst-kix_wu62sfo7w397-4>li {
    counter-increment: lst-ctn-kix_wu62sfo7w397-4
}
.lst-kix_rrwtpabby6sq-3>li:before {
    content: "\0025cf  "
}
.lst-kix_wu62sfo7w397-1>li:before {
    content: "" counter(lst-ctn-kix_wu62sfo7w397-1, lower-latin) ". "
}
.lst-kix_wu62sfo7w397-7>li {
    counter-increment: lst-ctn-kix_wu62sfo7w397-7
}
.lst-kix_rrwtpabby6sq-4>li:before {
    content: "\0025cb  "
}
.lst-kix_wu62sfo7w397-2>li:before {
    content: "" counter(lst-ctn-kix_wu62sfo7w397-2, lower-roman) ". "
}
.lst-kix_rrwtpabby6sq-1>li:before {
    content: "\0025cb  "
}
ul.lst-kix_5fbgyloudqbx-0 {
    list-style-type: none
}
ul.lst-kix_5fbgyloudqbx-2 {
    list-style-type: none
}
ul.lst-kix_5fbgyloudqbx-1 {
    list-style-type: none
}
ol.lst-kix_k47f4yk9x4ic-0.start {
    counter-reset: lst-ctn-kix_k47f4yk9x4ic-0 0
}
ol.lst-kix_k47f4yk9x4ic-6.start {
    counter-reset: lst-ctn-kix_k47f4yk9x4ic-6 0
}
.lst-kix_wu62sfo7w397-4>li:before {
    content: "" counter(lst-ctn-kix_wu62sfo7w397-4, lower-latin) ". "
}
ul.lst-kix_5fbgyloudqbx-4 {
    list-style-type: none
}
ul.lst-kix_5fbgyloudqbx-3 {
    list-style-type: none
}
.lst-kix_wu62sfo7w397-3>li:before {
    content: "" counter(lst-ctn-kix_wu62sfo7w397-3, decimal) ". "
}
.lst-kix_rrwtpabby6sq-2>li:before {
    content: "\0025a0  "
}
ul.lst-kix_5fbgyloudqbx-6 {
    list-style-type: none
}
ul.lst-kix_5fbgyloudqbx-5 {
    list-style-type: none
}
.lst-kix_k47f4yk9x4ic-2>li {
    counter-increment: lst-ctn-kix_k47f4yk9x4ic-2
}
.lst-kix_wu62sfo7w397-6>li:before {
    content: "" counter(lst-ctn-kix_wu62sfo7w397-6, decimal) ". "
}
ul.lst-kix_5fbgyloudqbx-8 {
    list-style-type: none
}
ul.lst-kix_5fbgyloudqbx-7 {
    list-style-type: none
}
ol.lst-kix_k47f4yk9x4ic-3.start {
    counter-reset: lst-ctn-kix_k47f4yk9x4ic-3 0
}
.lst-kix_wu62sfo7w397-5>li:before {
    content: "" counter(lst-ctn-kix_wu62sfo7w397-5, lower-roman) ". "
}
.lst-kix_rrwtpabby6sq-0>li:before {
    content: "\0025cf  "
}
.lst-kix_wu62sfo7w397-8>li {
    counter-increment: lst-ctn-kix_wu62sfo7w397-8
}
ul.lst-kix_rrwtpabby6sq-0 {
    list-style-type: none
}
ul.lst-kix_rrwtpabby6sq-1 {
    list-style-type: none
}
ul.lst-kix_rrwtpabby6sq-2 {
    list-style-type: none
}
ul.lst-kix_rrwtpabby6sq-3 {
    list-style-type: none
}
ul.lst-kix_rrwtpabby6sq-4 {
    list-style-type: none
}
.lst-kix_wu62sfo7w397-2>li {
    counter-increment: lst-ctn-kix_wu62sfo7w397-2
}
ul.lst-kix_rrwtpabby6sq-5 {
    list-style-type: none
}
ul.lst-kix_rrwtpabby6sq-6 {
    list-style-type: none
}
ul.lst-kix_rrwtpabby6sq-7 {
    list-style-type: none
}
ol.lst-kix_k47f4yk9x4ic-6 {
    list-style-type: none
}
ul.lst-kix_rrwtpabby6sq-8 {
    list-style-type: none
}
ol.lst-kix_k47f4yk9x4ic-7 {
    list-style-type: none
}
ol.lst-kix_k47f4yk9x4ic-8 {
    list-style-type: none
}
ol.lst-kix_k47f4yk9x4ic-7.start {
    counter-reset: lst-ctn-kix_k47f4yk9x4ic-7 0
}
.lst-kix_wu62sfo7w397-0>li:before {
    content: "" counter(lst-ctn-kix_wu62sfo7w397-0, decimal) ". "
}
ol.lst-kix_wu62sfo7w397-7.start {
    counter-reset: lst-ctn-kix_wu62sfo7w397-7 0
}
.lst-kix_wu62sfo7w397-1>li {
    counter-increment: lst-ctn-kix_wu62sfo7w397-1
}
.lst-kix_rrwtpabby6sq-7>li:before {
    content: "\0025cb  "
}
.lst-kix_rrwtpabby6sq-8>li:before {
    content: "\0025a0  "
}
.lst-kix_k47f4yk9x4ic-4>li {
    counter-increment: lst-ctn-kix_k47f4yk9x4ic-4
}
.lst-kix_k47f4yk9x4ic-7>li {
    counter-increment: lst-ctn-kix_k47f4yk9x4ic-7
}
ol.lst-kix_k47f4yk9x4ic-0 {
    list-style-type: none
}
ol.lst-kix_k47f4yk9x4ic-1 {
    list-style-type: none
}
ol.lst-kix_k47f4yk9x4ic-2 {
    list-style-type: none
}
ol.lst-kix_k47f4yk9x4ic-3 {
    list-style-type: none
}
.lst-kix_k47f4yk9x4ic-1>li {
    counter-increment: lst-ctn-kix_k47f4yk9x4ic-1
}
ol.lst-kix_k47f4yk9x4ic-4 {
    list-style-type: none
}
ol.lst-kix_k47f4yk9x4ic-1.start {
    counter-reset: lst-ctn-kix_k47f4yk9x4ic-1 0
}
ol.lst-kix_k47f4yk9x4ic-5 {
    list-style-type: none
}
ol.lst-kix_wu62sfo7w397-0.start {
    counter-reset: lst-ctn-kix_wu62sfo7w397-0 0
}
ol.lst-kix_wu62sfo7w397-6.start {
    counter-reset: lst-ctn-kix_wu62sfo7w397-6 0
}
ol.lst-kix_k47f4yk9x4ic-8.start {
    counter-reset: lst-ctn-kix_k47f4yk9x4ic-8 0
}
ol {
    margin: 0;
    padding: 0
}

.supp-c16 {
    padding: 5pt 5pt 5pt 5pt;

    outline: 0px;
    outline-left: 1px;
    vertical-align: middle;
    width: 234pt;
}
.supp-c18 {
    padding: 5pt 5pt 5pt 5pt;

    vertical-align: top;
    width: 234pt;
    outline: 0px;
}
.supp-c19 {
    height: 0pt;
    outline: 0px;
}
.supp-c26 {
    padding: 5pt 5pt 5pt 5pt;
    outline: 0px;
    vertical-align: top;
    width: 234pt;
}
.supp-c6 {
    color: #000000;
    font-weight: 400;
    text-decoration: none;
    vertical-align: baseline;
    font-size: 8pt;
    font-family: "Arial";
    font-style: normal
}
.supp-c11 {
    color: #000000;
    font-weight: 400;
    text-decoration: none;
    vertical-align: baseline;
    font-size: 9pt;
    font-family: "Arial";
    font-style: normal
}
.supp-c34 {
    color: #434343;
    font-weight: 400;
    text-decoration: none;
    vertical-align: baseline;
    font-size: 14pt;
    font-family: "Arial";
    font-style: normal
}
.supp-c4 {
    padding-top: 0pt;
    padding-bottom: 0pt;
    line-height: 1.15;
    orphans: 2;
    widows: 2;
    text-align: left;
    height: 11pt
}
.supp-c3x {
    color: #000000;
    font-weight: 400;
    text-decoration: none;
    vertical-align: baseline;
    font-size: 9pt;
    font-family: "Arial";
    font-style: normal
}
.supp-c3 {
    color: #000000;
    font-weight: 400;
    text-decoration: none;
    vertical-align: baseline;
    font-size: 11pt;
    font-family: "Arial";
    font-style: normal
}
.supp-c17 {
    padding-top: 0pt;
    padding-bottom: 0pt;
    line-height: 1.15;
    orphans: 2;
    widows: 2;
    text-align: center
}
.supp-c22 {
    padding-top: 18pt;
    padding-bottom: 6pt;
    line-height: 1.15;
    page-break-after: avoid;
    text-align: left;
    height: 16pt
}
.supp-c7 {
    padding-top: 0pt;
    padding-bottom: 0pt;
    line-height: 1.15;
    orphans: 2;
    widows: 2;
    text-align: left
}
.supp-c36 {
    padding-top: 0pt;
    padding-bottom: 0pt;
    line-height: 1.15;
    orphans: 2;
    widows: 2;
    text-align: right
}
.supp-c35 {
    font-weight: 400;
    text-decoration: none;
    vertical-align: baseline;
    font-size: 12pt;
    font-family: "Arial";
    font-style: normal
}
.supp-c28 {
    padding-top: 0pt;
    padding-bottom: 12pt;
    line-height: 1.15;
    orphans: 2;
    widows: 2;
    text-align: justify
}
.supp-c1 {
    padding-top: 0pt;
    padding-bottom: 0pt;
    line-height: 1.15;
    orphans: 2;
    widows: 2;
    text-align: justify
}
.supp-c30 {
    padding-top: 0pt;
    padding-bottom: 3pt;
    line-height: 1.15;
    page-break-after: avoid;
    text-align: left
}
.supp-c5 {
    padding-top: 0pt;
    padding-bottom: 0pt;
    line-height: 1.15;
    text-align: left;
    height: 11pt
}
.supp-c21 {
    padding-top: 18pt;
    padding-bottom: 6pt;
    line-height: 1.15;
    page-break-after: avoid;
    text-align: justify
}
.supp-c31 {
    padding-top: 18pt;
    padding-bottom: 6pt;
    line-height: 1.15;
    page-break-after: avoid;
    text-align: left
}
.supp-c15 {
    padding-top: 16pt;
    padding-bottom: 4pt;
    line-height: 1.15;
    page-break-after: avoid;
    text-align: justify
}
.supp-c2 {
    padding-top: 0pt;
    padding-bottom: 0pt;
    line-height: 1.0;
    text-align: left
}
.supp-c25 {
    padding-top: 12pt;
    padding-bottom: 12pt;
    line-height: 1.0;
    text-align: justify
}
.supp-c13 {
    vertical-align: baseline;
    font-size: 26pt;
    font-family: "Arial";
    font-style: normal
}
.supp-c14 {
    vertical-align: baseline;
    font-size: 16pt;
    font-family: "Arial";
    font-style: normal
}
.supp-c33 {
    border-spacing: 0;
    border-collapse: collapse;
    margin: auto;
    border: 0px;
    outline: 0px;
}
.supp-c29 {
    vertical-align: baseline;
    font-size: 11pt;
    font-family: "Arial"
}
.supp-c20 {
    background-color: #ffffff;
    max-width: 468pt;
    padding: 72pt 72pt 72pt 72pt
}
.supp-c32 {
    font-size: 10pt;
    font-family: "Courier New";
    font-weight: 400
}
.supp-c37 {
    font-weight: 400;
    text-decoration: none;
    font-style: normal
}
.supp-c12 {
    color: #000000;
    font-weight: 400;
    text-decoration: none
}
.supp-c9 {
    color: inherit;
    text-decoration: inherit
}
.supp-c0 {
    color: #1155cc;
    text-decoration: underline
}
.supp-c10 {
    font-size: 9pt;
    font-weight: 700
}
.supp-c24 {
    height: 11pt
}
.supp-c8 {
    font-style: italic
}
.supp-c23 {
    background-color: #ffffff
}
.supp-c27 {
    color: #24292e
}

</style>


</head>
<body class="c24">
   <div>
      <p class="c17 c9"><span class="c1"></span></p>
   </div>
   <h1 class="c14" id="h.bx85djk6ygxo"><span>HiGlass</span><span class="c13 c5">: Web-based visual comparison and exploration of genome interaction maps</span></h1>
   <p class="c17 c9"><span class="c1"></span></p>
   <p class="c17"><span>Peter Kerpedjiev</span><span class="c4">1</span><span>, Nezar Abdennur</span><span class="c4">2</span><span>, Fritz Lekschas</span><span class="c4">3</span><span>, Chuck McCallum</span><span class="c4">1</span><span>, Kasper Dinkla</span><span class="c4">3</span><span>, Hendrik Strobelt</span><span class="c4">3</span><span>, Jacob M Luber</span><span class="c4">1,4</span><span>, Scott Ouellette</span><span class="c4">1</span><span>, Alaleh Azhir</span><span class="c4">1</span><span>, Nikhil Kumar</span><span class="c4">1</span><span>, Jeewon Hwang</span><span class="c4">1</span><span>, Burak H Alver</span><span class="c4">1</span><span>, Hanspeter Pfister</span><span class="c4">3</span><span>, Leonid A Mirny</span><span class="c4">5,6</span><span>, Peter J Park</span><span class="c4">1</span><span>, Nils Gehlenborg</span><span class="c4">1,</span><span>*</span></p>
   <p class="c9 c17"><span class="c1"></span></p>
   <p class="c17"><span class="c3 c4">1</span><span class="c3">&nbsp;Department of Biomedical Informatics, </span><span class="c3">Harvard</span><span class="c2">&nbsp;Medical School, Boston, MA, USA </span></p>
   <p class="c17"><span class="c3 c4">2</span><span class="c2">&nbsp;Computational and Systems Biology Program, MIT, Cambridge, USA </span></p>
   <p class="c17"><span class="c3 c4">3</span><span class="c2">&nbsp;School of Engineering and Applied Sciences, Harvard University, Cambridge, MA, USA</span></p>
   <p class="c17"><span class="c3 c4">4</span><span class="c3">&nbsp;Bioinformatics and Integrative Genomics Program, Harvard Medical School, Boston, MA, USA</span></p>
   <p class="c17"><span class="c3 c4">5</span><span class="c2">&nbsp;Department of Physics, MIT, Cambridge, USA</span></p>
   <p class="c17"><span class="c3 c4">6</span><span class="c2">&nbsp;Institute for Medical Engineering and Science, MIT, Cambridge, USA</span></p>
   <p class="c17 c9"><span class="c2"></span></p>
   <p class="c17"><span class="c3 c10">* Correspondence should be addressed to </span><span class="c3 c10 c25 c19"><a class="c0" href="mailto:nils@hms.harvard.edu">nils@hms.harvard.edu</a></span><span class="c3 c10">.</span></p>
   <h3 class="c31" id="h.ps9jld4z5myl"><span class="c30"></span></h3>
   <p class="c6"><span class="c10">We present HiGlass (</span><span class="c10 c25 c19"><a class="c0" href="https://www.google.com/url?q=http://higlass.io&amp;sa=D&amp;ust=1497805615900000&amp;usg=AFQjCNFnevwKo0QInWHPY4yyEVaKLFAjCA">http://higlass.io</a></span><span class="c10">), a web-based viewer for genome interaction </span><span class="c10">map</span><span class="c10">s featuring synchronized navigation of multiple views as well as </span><span class="c10">continuous</span><span class="c10">&nbsp;zooming and panning for navigation across genomic loci and resolutions. We demonstrate how our implementation of visual comparison for Hi-C and other genomic data from different experimental </span><span class="c10">conditions</span><span class="c10">&nbsp;can </span><span class="c10">be used</span><span class="c10">&nbsp;to efficiently identify salient outcomes of experimental perturbations, generate new hypotheses</span><span class="c10">,</span><span class="c10">&nbsp;and share the results with the community.</span></p>
   <p class="c6 c9"><span class="c1"></span></p>
   <p class="c17 c9"><span class="c1"></span></p>
   <p class="c17 c9"><span class="c1"></span></p>
   <p class="c6"><span>The development of chromosome capture assays measuring the proximity of two or more non-adjacent regions of the genome</span><span>&nbsp;helped elucidate how t</span><span>he</span><span>&nbsp;structure and </span><span>dynamics </span><span>of chromosome structure affect</span><span>&nbsp;gene regulation and cellular function</span><span class="c4 c5">1,2</span><span>. Techniques such as Hi-C have </span><span>revealed</span><span>&nbsp;features of genome organization such as A/B compartments, topologically associated domains (TADs) and loops</span><span class="c5 c4">1,3&ndash;5</span><span>. They have helped implicate changes in genome organization in a variety of disorders, including acute lymphoblastic leukemia</span><span class="c5 c4">6</span><span>, colorectal cancer</span><span class="c5 c4">7</span><span>, and limb development disorders</span><span class="c5 c4">8</span><span>. More fundamentally, they provide </span><span>insights into the mechanisms of how genome conformation structures arise and are maintained</span><span class="c5 c4">9&ndash;11</span><span>. Major efforts like the 4D Nucleome Network and the ENCODE project are generating such data at large scale across different cell lines and conditions with the aim of understand</span><span>ing</span><span>&nbsp;the mechanisms that govern processes such as gene regulation and DNA replication as well as to cross-validate the results from different experimental assays</span><span class="c5 c4">2,12</span><span class="c5 c4">13</span><span class="c5 c4">2,12</span><span>.</span></p>
   <p class="c6 c9"><span class="c1"></span></p>
   <p class="c6"><span>To</span><span>&nbsp;obtain genome conformation capture maps, raw sequencing data are processed to identify captured proximity ligation events such as contacts or interactions between genomic loci, which are then binned to form contact matrices</span><span class="c5 c4">14&ndash;16</span><span>; see </span><span>Lajoie et al.</span><span class="c5 c4">17</span><span>&nbsp;and Ay &amp; Noble</span><span class="c5 c4">18</span><span>&nbsp;</span><span>for reviews of Hi-C data processing. The d</span><span>iscovery of novel genomic structures and mechanisms, however, also requires sophisticated visual tools </span><span>for exploring features relevant at scales ranging from tens to millions of base pairs</span><span class="c5 c4">15,19,20</span><span>. Given the multiscale features of genome organization</span><span>,</span><span>&nbsp;it is crucial that such visualization tools support </span><span>continuous zooming and panning, </span><span>comparison across multiple scales and conditions as well as integration with additional genomic and epigenomic data.</span><span>&nbsp;</span><span>E</span><span>xisting tools </span><span>support</span><span>&nbsp;different ways of displaying contact frequencies</span><span>, such as </span><span>rectangular heatmaps</span><span>, triangular heatmaps, arc plots,</span><span>&nbsp;or</span><span>&nbsp;circular plots, </span><span>and </span><span>different levels of interaction</span><span>&nbsp;ranging from static plotting to interactive zooming and panning, as well as different levels of </span><span>integration with other genomic data </span><span>types</span><span class="c5 c4">15,21&ndash;24</span><span>. </span><span>None of the existing tools</span><span>, h</span><span>owever, </span><span>provides</span><span>&nbsp;</span><span>flexible methods for </span><span>fast, </span><span>synchronized</span><span>&nbsp;navigation across multiple conditions, resolutions and l</span><span>oci or the ability to share interactive visualizations with the broader community.</span><span>&nbsp;</span></p>
   <p class="c6 c9"><span class="c1"></span></p>
   <p class="c6"><span>To address these shortcomings,</span><span>&nbsp;we created HiGlass, an open source, web-based application with a customizable interface designed to support</span><span>&nbsp;</span><span>multiscale contact map visualization across multiple resolutions, loci, and conditions (<a href="#supp-methods">Supplementary Methods</a>).</span><span>&nbsp;</span><span>HiGlass introduces the concept of </span><span class="c10">composable linked views</span><span class="c10">&nbsp;</span><span>for genomic data visualization (<a href="#supp-fig1">Supplementary Fig. 1</a>)</span><span class="c10">.</span><span>&nbsp;This approach enables users to compose the layout, content, and synchronization of locus, zoom-level, and other view properties across multiple views (<a href="#supp-fig2">Supplementary Fig. 2</a>). </span><span>These view compositions can range from the juxtaposition of two or more heatmaps (<a href="#figure1">Fig. 1</a>) to sophisticated collations of matrices, tracks and viewport projections (<a href="#figure2">Fig. 2</a>). </span><span>View compositions serve to display data at multiple scales, </span><span>to</span><span>&nbsp;corroborate </span><span>observations </span><span>with other types of evidence and to facilitate comparison</span><span>s</span><span>&nbsp;between experiments. </span><span>As</span><span>&nbsp;a web-based tool, HiGlass also supports storing and sharing of view compositions with other investigators and the public via hyper</span><span>links</span><span>. The tool can be used to access selected public datasets at </span><span class="c19"><a class="c0" href="http://higlass.io">http://higlass.io</a></span><span>, or </span><span>it may be </span><span>run locally and populated with private data using </span><span>a</span><span class="c1">&nbsp;provided Docker container.</span></p>
   <p class="c6 c9"><span class="c1"></span></p>

   <div>
       <div style="position: relative">
       <div style="padding: 10px; border: 1px solid black; position: relative;">
           <span class="fullscreen"><span class="c3 c19 c25"><a class="c0" href="http://higlass.io/app/?config=Tf2-ublRTey9hiBKMlgzwg">Open in fullscreen</a></span></span>
         <div>
             <div id="figure1" style="height: 800px" />
        </div>
         </div>
             
<!--
   <p class="c6">

   <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 477.33px;">

   <img alt="" src="assets/images/image1.png" style="width: 624.00px; height: 477.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
   <p class="c6 c9"><span class="c7 c29">
   	

   </span>
   </p>
   -->


   <p class="c6"><span class="c23 c3">Figure 1</span><span class="c23 c3">&nbsp;|</span><span class="c3">&nbsp;</span><span class="c3">An interactive view composition highlighting the results from Schwarzer et al. </span><span class="c3">with data from WT</span><span class="c3">&nbsp;(left), and mutant</span><span class="c3">&nbsp;(&Delta;Nipbl, right) samples</span><span class="c3 c4">25</span><span class="c3">. The top two views are linked to each other by zoom and location such that they always display the same region at the same resolution. Comparing the control (left) and mutant (right) condition at this zoom level reveals the bleaching of TADs in the gene-poor region in the lower right hand part of the maps. The bottom two views, also linked to each other by zoom and location, show a zoomed in perspective where a more fragmented compartmentalization of the </span><span class="c3">&Delta;Nipbl</span><span class="c3 c10">&nbsp;</span><span class="c3">mutant</span><span class="c3">&nbsp;(right) as compared to WT (left) can be seen. The black rectangles in the top views, which are referred to as viewport projections in HiGlass, show the positions and extent of the bottom views (<a href="supp-fig3">Supplementary Fig. 3</a>). The white lines in the bottom left panel are a result of bins filtered during matrix balancing. An full-screen version of this figure is available at </span><span class="c3 c19 c25"><a class="c0" href="http://higlass.io/app/?config=Tf2-ublRTey9hiBKMlgzwg">http://higlass.io/app/?config=Tf2-ublRTey9hiBKMlgzwg</a></span><span class="c3">.</span></p>
             </div>
    </div>

   <p class="c6 c9"><span class="c1"></span></p>
   <p class="c6"><span>To illustrate the utility of composable linked views, we used HiGlass to highlight key results of a recent study showing the effect of induced deletion of the cohesin loading factor </span><span class="c10">Nibpl</span><span>&nbsp;on chromosome organization in adult mouse hepatocytes</span><span class="c5 c4">25</span><span>. We </span><span>obtained</span><span>&nbsp;Hi-C contact data and binned it at multiple resolutions starting at</span><span>&nbsp;1 kb</span><span>&nbsp;</span><span>for</span><span>&nbsp;wild type (WT) and &Delta;Nipbl primary hepatocytes (<a href="#supp-methods">Supplementary Methods</a>). We loaded both samples as separate views (</span><span><a href="#figure1">Fig. 1</a></span><span>, top) and</span><span>&nbsp;</span><span>linked them via location and zoom level</span><span>.</span><span>&nbsp;</span><span>With t</span><span>he</span><span>&nbsp;two linked views, we could navigate to regions clearly showing the disappearance of features in the &Delta;Nipbl condition. For example, in the gene-poor region from chr14:80 Mb to chr14:100 Mb of mm9, we observe a robust &ldquo;bleaching&rdquo; of near-diagonal contact enrichment patterns. The patterns that disappeared are examples of TADs because they do not show the long range checkering characteristic of A/B compartments.</span><span>&nbsp;In contrast, in the relatively gene-rich region </span><span>upstream of chr14:80 Mb</span><span>, we see an enhancement of the checkered pattern and the emergence of a finer division of A/B regions in the &Delta;Nipbl condition. </span><span>To explore this region more closely, we created two additional linked views for WT and &Delta;Nipbl and navigated to the region between chr14:50 Mb and chr14:70 Mb</span><span>&nbsp;(<a href="#figure1">Fig. 1</a>, bottom)</span><span>.</span><span>&nbsp;</span><span>Adding H3K4me3 and H3K27ac ChIP-seq signal tracks revealed that these marks, while similar between conditions, correlate more strongly with the compartmentalization pattern in &Delta;Nipbl. </span><span>Finally, we used a viewport projection to mark the position of the bottom views relative to the top, resulting in the complete view composition shown in <a href="#figure1">Fig. 1</a>. This</span><span>&nbsp;interactive visual </span><span>recapitulation of key</span><span>&nbsp;results</span><span>&nbsp;from Schwarzer et al</span><span>&nbsp;illustrates how synchronized navigation across loci and </span><span>resolutions</span><span>&nbsp;by linking views between multiple conditions facilitates the exploration of the complex effects of global perturbations on chromosome organization at multiple scales.</span></p>
   <p class="c6 c9"><span class="c1"></span></p>
   

<!--
<p class="c6">
   <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 426.67px;">




   <img alt="" src="assets/images/image2.png" style="width: 624.00px; height: 426.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""> 



   </span>
      </p>

   -->
   <div>
       <div style="position: relative">
       <div style="padding: 10px; border: 1px solid black; position: relative;">
           <span class="fullscreen"><span class="c3 c19 c25"><a class="c0" href="http://higlass.io/app/?config=Q5LdNchQRLSZ_0yKsTEoiw">Open in fullscreen</a></span></span>
         <div>
             <div id="figure2" style="height: 800px" />
        </div>
         </div>


   <p class="c6 c9"><span class="c1"></span></p>
   <p class="c6"><span class="c23 c3">Figure 2</span><span class="c3 c23">&nbsp;|</span><span class="c3">&nbsp;An interactive view composition containing two views linked by location and zoom (top) and an independent (unlinked) zoomed out overview (bottom) (<a href="supp-fig4">Supplementary Fig. 4</a>). The two views on top show data from chromosome 14 (mm9) in the wild type and &Delta;Nipbl</span><span class="c3 c10">&nbsp;</span><span class="c3">conditions, respectively. The bottom view shows data from the mutant condition as well as a projection of the viewport visible in the top views. The patch visible in the &Delta;Nipbl</span><span class="c3 c10">&nbsp;</span><span class="c3">condition (top left) is notably absent from the control (right). The gene annotations, RNAseq, H3K27me and H3K4me3 tracks show the p</span><span class="c3">resence and transcription of the </span><span class="c3 c10">Dock5</span><span class="c3">&nbsp;and </span><span class="c3 c10">Mycbp2</span><span class="c3">&nbsp;genes on the - strand as well as the presence and transcription of the </span><span class="c3 c10">Gnrh1</span><span class="c3">&nbsp;and </span><span class="c3 c10">Cln5</span><span class="c3">&nbsp;genes on the + </span><span class="c3">strand</span><span class="c3">. </span><span class="c3">An interactive version of this figure is available at</span><span class="c3">&nbsp;</span><span class="c3 c25 c19"><a class="c0" href="http://higlass.io/app/?config=Q5LdNchQRLSZ_0yKsTEoiw">http://higlass.io/app/?config=Q5LdNchQRLSZ_0yKsTEoiw</a></span><span class="c3">.</span></p>
       </div>
       </div>
   <p class="c6 c9"><span class="c1"></span></p>
   <p class="c6"><span>Using the same view composition</span><span>&nbsp;we noticed the appearance of </span><span>small dark patches away from the diagonal</span><span>. To investigate them we created a new composition containing an overview and two zoom- and location-linked detail </span><span>views</span><span>&nbsp;</span><span>(<a href="#figure2">Fig. 2</a>)</span><span>. By using the overview to find patches and comparing them using the detail views,</span><span>&nbsp;we established that</span><span>&nbsp;they are more </span><span>enriched in the mutant condition than in the wild type, that they represent interaction</span><span>&nbsp;between pairs of short active regions (type A compartment) and that they are closely aligned with gene annotations.</span><span>&nbsp;</span><span>Including RNA-seq and ChIP-seq tracks let us see that the genes which align with these patches are virtually always transcriptionally active</span><span class="c1">. This further supports the conclusion of Schwarzer et al. that the absence of cohesin loading leads to strengthened A/B compartmentalization. It also provides avenues for further investigation and analysis of the structural and regulatory consequences of cohesin loss.</span></p>
   <p class="c6 c9"><span class="c1"></span></p>
   <p class="c6"><span>Using HiGlass to create the composable linked views shown in <a href="#figure1">Figs. 1</a> and <a href="#figure2">2</a> enabled us to interactively explore the data generated by Schwarzer et al. across different conditions, zoom levels and loci. This gave us not only a clearer understanding of the results, but allowed us to see unexpected patterns and rapidly gather observations to be used in generating new hypotheses. We </span><span>provide</span><span>d links to interactive online versions of the figures to disseminate the findings and encourage collaboration and discovery. Our public demo (</span><span class="c25 c19"><a class="c0" href="http://higlass.io">http://higlass.io</a></span><span>) contains additional examples showing how HiGlass can be used as a performant standalone genome browser to display 1D genomic data</span><span class="c5 c4">26</span><span>&nbsp;while simultaneously providing the same view composition operations described here. </span><span class="c1">Going forward, we aim to extend HiGlass to support not only additional data types but also new ways to collaboratively explore, annotate and analyze large genomic data sets.</span></p>
   <h3 class="c22" id="h.qn35prkf71d1"><span class="c5 c27">Author Contributions</span></h3>
   <p class="c6 c9"><span class="c7 c29"></span></p>
   <p class="c6"><span class="c1">PK and NG conceived the research. PK, NA and NG wrote the manuscript with input from LAM, PJP, and BHA. PK, NA, FL, and CM wrote the software with help from KD, HS, JML, SO, AA, NK, and JH. BHA, HP, LAM, and PJP provided valuable input and advice for the project.</span></p>
   <h3 class="c22" id="h.m2s3dqme6fqs"><span class="c30">Acknowledgements</span></h3>
   <p class="c6 c9"><span class="c1"></span></p>
   <p class="c6"><span>We thank Francois Spitz, Wibke Schwarzer, and Aleksandra Pekowska for providing the data presented in this paper. We also acknowledge important </span><span>suggestions and feedback from</span><span class="c1">&nbsp;members of the Park Lab at Harvard Medical School, the Mirny Lab at MIT, and the Dekker Lab at University of Massachusetts Medical School as well as members of 4D Nucleome Data Coordination and Integration Center who provided input and feedback. This project was made possible by funding from the National Institutes of Health (U01 CA200059, R00 HG007583, and U54 HG007963).</span></p>
   <h1 class="c20" id="h.yratt83t4j42"><span class="c5 c13">References</span></h1>



   <p class="c28"><span class="c8 c12">1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Lieberman-Aiden, E. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Comprehensive mapping of long-range interactions reveals folding principles of the human genome. </span><span class="c5 c10 c11">Science</span><span class="c5 c8">&nbsp;</span><span class="c7">326,</span><span class="c5 c8">&nbsp;289&ndash;293 (2009).</span></p>
   <p class="c15"><span class="c8 c12">2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Dekker, J., Marti-Renom, M. A. &amp; Mirny, L. A. Exploring the three-dimensional organization of genomes: interpreting chromatin interaction data. </span><span class="c5 c10 c11">Nat. Rev. Genet.</span><span class="c5 c8">&nbsp;</span><span class="c7">14,</span><span class="c5 c8">&nbsp;390&ndash;403 (2013).</span></p>
   <p class="c15"><span class="c8 c12">3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Dixon, J. R. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Topological domains in mammalian genomes identified by analysis of chromatin interactions. </span><span class="c5 c10 c11">Nature</span><span class="c5 c8">&nbsp;</span><span class="c7">485,</span><span class="c5 c8">&nbsp;376&ndash;380 (2012).</span></p>
   <p class="c15"><span class="c8 c12">4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Nora, E. P. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Spatial partitioning of the regulatory landscape of the X-inactivation centre. </span><span class="c5 c10 c11">Nature</span><span class="c5 c8">&nbsp;</span><span class="c7">485,</span><span class="c5 c8">&nbsp;381&ndash;385 (2012).</span></p>
   <p class="c15"><span class="c8 c12">5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Rao, S. S. P. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;A 3D Map of the Human Genome at Kilobase Resolution Reveals Principles of Chromatin Looping. </span><span class="c5 c10 c11">Cell</span><span class="c5 c8">&nbsp;</span><span class="c7">159,</span><span class="c5 c8">&nbsp;1665&ndash;1680 (2014).</span></p>
   <p class="c15"><span class="c8 c12">6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Hnisz, D. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Activation of proto-oncogenes by disruption of chromosome neighborhoods. </span><span class="c5 c10 c11">Science</span><span class="c5 c8">&nbsp;</span><span class="c7">351,</span><span class="c5 c8">&nbsp;1454&ndash;1458 (2016).</span></p>
   <p class="c15"><span class="c8 c12">7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Seaman, L. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Nucleome Analysis Reveals Structure-function Relationships for Colon Cancer. </span><span class="c5 c10 c11">Mol. Cancer Res.</span><span class="c5 c8">&nbsp;(2017). doi:</span><span class="c5 c8">10.1158/1541-7786.MCR-16-0374</span></p>
   <p class="c15"><span class="c8 c12">8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Lupi&aacute;&ntilde;ez, D. G. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Disruptions of Topological Chromatin Domains Cause Pathogenic Rewiring of Gene-Enhancer Interactions. </span><span class="c5 c10 c11">Cell</span><span class="c5 c8">&nbsp;</span><span class="c7">161,</span><span class="c5 c8">&nbsp;1012&ndash;1025 (2015).</span></p>
   <p class="c15"><span class="c8 c12">9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Sanborn, A. L. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. </span><span class="c5 c10 c11">Proc. Natl. Acad. Sci. U. S. A.</span><span class="c5 c8">&nbsp;</span><span class="c7">112,</span><span class="c5 c8">&nbsp;E6456&ndash;65 (2015).</span></p>
   <p class="c15"><span class="c8 c12">10.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Fudenberg, G. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Formation of Chromosomal Domains by Loop Extrusion. </span><span class="c5 c10 c11">Cell Rep.</span><span class="c5 c8">&nbsp;</span><span class="c7">15,</span><span class="c5 c8">&nbsp;2038&ndash;2049 (2016).</span></p>
   <p class="c15"><span class="c8 c12">11.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Zuin, J. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Cohesin and CTCF differentially affect chromatin architecture and gene expression in human cells. </span><span class="c5 c10 c11">Proc. Natl. Acad. Sci. U. S. A.</span><span class="c5 c8">&nbsp;</span><span class="c7">111,</span><span class="c5 c8">&nbsp;996&ndash;1001 (2014).</span></p>
   <p class="c15"><span class="c8 c12">12.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Dekker, J. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;The 4D Nucleome Project. </span><span class="c5 c10 c11">bioRxiv</span><span class="c5 c8">&nbsp;103499 (2017). doi:</span><span class="c5 c8">10.1101/103499</span></p>
   <p class="c15"><span class="c8 c12">13.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. </span><span class="c5 c10 c11">Nature</span><span class="c5 c8">&nbsp;</span><span class="c7">489,</span><span class="c5 c8">&nbsp;57&ndash;74 (2012).</span></p>
   <p class="c15"><span class="c8 c12">14.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Imakaev, M. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Iterative correction of Hi-C data reveals hallmarks of chromosome organization. </span><span class="c5 c10 c11">Nat. Methods</span><span class="c5 c8">&nbsp;</span><span class="c7">9,</span><span class="c5 c8">&nbsp;999&ndash;1003 (2012).</span></p>
   <p class="c15"><span class="c8 c12">15.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Durand, N. C. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Juicebox Provides a Visualization System for Hi-C Contact Maps with Unlimited Zoom. </span><span class="c5 c10 c11">Cell Syst</span><span class="c5 c8">&nbsp;</span><span class="c7">3,</span><span class="c5 c8">&nbsp;99&ndash;101 (2016).</span></p>
   <p class="c15"><span class="c8 c12">16.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Servant, N. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;HiC-Pro: an optimized and flexible pipeline for Hi-C data processing. </span><span class="c5 c10 c11">Genome Biol.</span><span class="c5 c8">&nbsp;</span><span class="c7">16,</span><span class="c5 c8">&nbsp;259 (2015).</span></p>
   <p class="c15"><span class="c8 c12">17.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Lajoie, B. R., Dekker, J. &amp; Kaplan, N. The Hitchhiker&rsquo;s guide to Hi-C analysis: practical guidelines. </span><span class="c5 c10 c11">Methods</span><span class="c5 c8">&nbsp;</span><span class="c7">72,</span><span class="c5 c8">&nbsp;65&ndash;75 (2015).</span></p>
   <p class="c15"><span class="c8 c12">18.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Ay, F. &amp; Noble, W. S. Analysis methods for studying the 3D architecture of the genome. </span><span class="c5 c10 c11">Genome Biol.</span><span class="c5 c8">&nbsp;</span><span class="c7">16,</span><span class="c5 c8">&nbsp;183 (2015).</span></p>
   <p class="c15"><span class="c8 c12">19.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Gehlenborg, N. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Visualization of omics data for systems biology. </span><span class="c5 c10 c11">Nat. Methods</span><span class="c5 c8">&nbsp;</span><span class="c7">7,</span><span class="c5 c8">&nbsp;S56&ndash;68 (2010).</span></p>
   <p class="c15"><span class="c8 c12">20.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Nielsen, C. B., Cantor, M., Dubchak, I., Gordon, D. &amp; Wang, T. Visualizing genomes: techniques and challenges. </span><span class="c5 c10 c11">Nat. Methods</span><span class="c5 c8">&nbsp;</span><span class="c7">7,</span><span class="c5 c8">&nbsp;S5&ndash;S15 (2010).</span></p>
   <p class="c15"><span class="c8 c12">21.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Wang, Y. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;The 3D Genome Browser: a web-based browser for visualizing 3D genome organization and long-range chromatin interactions. </span><span class="c5 c10 c11">bioRxiv</span><span class="c5 c8">&nbsp;112268 (2017). doi:</span><span class="c5 c8">10.1101/112268</span></p>
   <p class="c15"><span class="c8 c12">22.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Zhou, X. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Exploring long-range genome interactions using the WashU Epigenome Browser. </span><span class="c5 c10 c11">Nat. Methods</span><span class="c5 c8">&nbsp;</span><span class="c7">10,</span><span class="c5 c8">&nbsp;375&ndash;376 (2013).</span></p>
   <p class="c15"><span class="c8 c12">23.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Akdemir, K. C. &amp; Chin, L. HiCPlotter integrates genomic data with interaction matrices. </span><span class="c5 c10 c11">Genome Biol.</span><span class="c5 c8">&nbsp;</span><span class="c7">16,</span><span class="c5 c8">&nbsp;198 (2015).</span></p>
   <p class="c15"><span class="c8 c12">24.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Yard&#305;mc&#305;, G. G. &amp; Noble, W. S. Software tools for visualizing Hi-C data. </span><span class="c5 c10 c11">Genome Biol.</span><span class="c5 c8">&nbsp;</span><span class="c7">18,</span><span class="c5 c8">&nbsp;26 (2017).</span></p>
   <p class="c15"><span class="c8 c12">25.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Schwarzer, W. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Two independent modes of chromosome organization are revealed by cohesin removal. </span><span class="c5 c10 c11">bioRxiv</span><span class="c5 c8">&nbsp;094185 (2016). doi:</span><span class="c5 c8">10.1101/094185</span></p>
   <p class="c16"><span class="c8 c12">26.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c5 c8">Busslinger, G. A. </span><span class="c5 c10 c11">et al.</span><span class="c5 c8">&nbsp;Cohesin is positioned in mammalian genomes by transcription, CTCF and Wapl. </span><span class="c5 c10 c11">Nature</span><span class="c5 c8">&nbsp;</span><span class="c7">544,</span><span class="c5 c8">&nbsp;503&ndash;507 (2017).</span></p>

   <hr style="margin-bottom: 40px; margin-top: 40px;" /> 

   <p class="supp-c30 title" id="h.ku9exhxrfp6z"><span class="supp-c12 c13">Supplementary Information</span></p>
   <h2 class="supp-c31" id="h.bt4pdtb535u2"><span class="supp-c14 c12">Supplementary Figures</span></h2>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <div id="supp-fig1" />
   <p class="supp-c7"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 445.33px;"><img alt="" src="assets/images/paper_supplement/image6.png" style="width: 624.00px; height: 445.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
   <p class="supp-c1"><span class="supp-c10">Supplementary Figure 1 |</span><span class="supp-c11">&nbsp;Views and view composition. A view composition in HiGlass consists of a spatial arrangement of views, the data tracks within each view and the links that relate views between each other. In this figure, the top two views are linked by location and zoom so that whenever the user pans or zoom in one of them, the other automatically follows. This form of link only becomes noticeable with user interaction. Another type of link, the mirrored viewport, ensures that the visible area of one view is visible as an outline on another. In this example, the top two views mirror their viewports on the bottom two views.</span></p>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <div id="supp-fig2" />
   <p class="supp-c17"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 509.00px; height: 479.32px;"><img alt="" src="assets/images/paper_supplement/image7.png" style="width: 509.00px; height: 479.32px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <p class="supp-c1"><span class="supp-c10">Supplementary Figure 2 |</span><span class="supp-c11">&nbsp;Examples of different ways that views can be linked. Multiple views of the same (b) or different datasets (a,c,d) can be composed and linked to facilitate data exploration and comparison. Two independent views of different samples provide free and independent exploration of each sample (a). Linking by zoom and location enforces the same scale and location in both samples (c). Zoom linking maintains the same scale while allowing free independent manipulation of the location (b). By linking location and leaving zooming free, one set of views can show an overview of a high resolution region (d). Displaying the extent of one view in another shows where a detail view is located in an overview (b,d). </span></p>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <p class="supp-c17 c24"><span class="supp-c3"></span></p>
   <div id="supp-fig3" />
   <p class="supp-c7"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 584.00px; height: 239.00px;"><img alt="" src="assets/images/paper_supplement/image5.png" style="width: 584.00px; height: 239.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <p class="supp-c1"><span class="supp-c10">Supplementary Figure 3</span><span class="supp-c11">&nbsp;| Schematic of the two sample / two location / two zoom level comparison shown in Fig. 1. The two views on top always show the same location at the same zoom level, as do the two on the bottom. Because bottom views are usually (but not necessarily) more zoomed in than the top, the viewport projection shows their location in a subset of the region shown on top.</span></p>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <hr style="page-break-before:always;display:none;">
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <div id="supp-fig4" />
   <p class="supp-c17"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 466.00px; height: 261.00px;"><img alt="" src="assets/images/paper_supplement/image3.png" style="width: 466.00px; height: 261.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
   <p class="supp-c1"><span class="supp-c10">Supplementary Figure 4</span><span class="supp-c11">&nbsp;| Schematic of the exploratory view. This schematic describes the linking present in Fig. 2. The top two views are linked by zoom and location showing the same genomic position across two samples. The bottom view shows the locations of the upper two but is controlled independently (not linked). </span></p>
   <h2 class="supp-c22" id="h.fdf5ezviih7j"><span class="supp-c14 c12"></span></h2>
   <hr style="page-break-before:always;display:none;">

   <div id="supp-methods" />
   <h2 class="supp-c31" id="h.ffkxm6yf7d5p"><span class="supp-c12 c14">Supplementary Methods</span></h2>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <p class="supp-c1"><span>In addition to the sections below, there is extensive documentation on the usage, capabilities and architecture of HiGlass available at </span><span class="supp-c0"><a class="supp-c9" href="https://github.com/hms-dbmi/higlass/wiki">https://github.com/hms-dbmi/higlass/wiki</a></span><span class="supp-c3">.</span></p>
   <h3 class="supp-c15" id="h.d7sztgrtvg9v"><span class="supp-c34">Architecture</span></h3>
   <p class="supp-c1"><span class="supp-c3">HiGlass is designed as a client-server application. Large data files are stored on a server and accessed in a piecewise fashion by the client. At any given position and zoom level, the client calculates which portion of the data it needs to display in that region. This portion of data is represented as a set of tiles, or discrete fixed-size chunks of data. The client requests this set of tiles from the server and awaits a reply. When it receives the reply with the data contained within the tiles, it renders it in its designated place. This makes it possible to quickly and uninterruptedly zoom and pan through a large dataset that would otherwise be impossible to load in its entirety. It should be noted that this almost identical to the manner in which online maps are loaded and rendered in web browsers.</span></p>
   <p class="supp-c1 c24"><span class="supp-c3"></span></p>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <a id="t.ac43f60ceb5eb051787d662a9e8ee9db5c643df7"></a><a id="t.0"></a>
   <table class="supp-c33">
      <tbody>
         <tr class="supp-c19">
            <td class="supp-c18" colspan="1" rowspan="1">
               <p class="supp-c7"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 253.50px; height: 201.88px;"><img alt="" src="assets/images/paper_supplement/image4.png" style="width: 253.50px; height: 201.88px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
            </td>
            <td class="supp-c16" colspan="1" rowspan="1">
               <p class="supp-c2"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 297.00px; height: 29.33px;"><img alt="" src="assets/images/paper_supplement/image1.png" style="width: 297.00px; height: 29.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
               <p class="supp-c2 supp-c24"><span class="supp-c3"></span></p>
               <p class="supp-c2 supp-c24"><span class="supp-c3"></span></p>
               <p class="supp-c2"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 297.00px; height: 28.00px;"><img alt="" src="assets/images/paper_supplement/image2.png" style="width: 297.00px; height: 28.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
            </td>
         </tr>
         <tr class="supp-c19">
            <td class="supp-c18" colspan="1" rowspan="1">
               <p class="supp-c2"><span class="supp-c3x">An example of the subdivision of a heatmap into tiles. The first digit corresponds to the zoom level while the second and third correspond to the x and y position of the tile.</span></p>
            </td>
            <td class="supp-c26" colspan="1" rowspan="1">
               <p class="supp-c2"><span class="supp-c3x">Other types of tracks such as lines and gene annotations are similarly divided into tiles and loaded in a piecewise fashion. Because these data are displayed along one dimension, they only require a zoom level and x coordinate.</span></p>
            </td>
         </tr>
      </tbody>
   </table>
   <p class="supp-c28 supp-c24"><span class="supp-c3"></span></p>
   <p class="supp-c28"><span>We provide more detailed technical and conceptual information on the higlass wiki under the &ldquo;Tiles and tiling section&rdquo;: </span><span class="supp-c0"><a class="supp-c9" href="https://github.com/hms-dbmi/higlass/wiki/Tiles-and-tiling">https://github.com/hms-dbmi/higlass/wiki/Tiles-and-tiling</a></span><span class="supp-c23 supp-c27 supp-c35">.</span></p>
   <p class="supp-c28"><span class="supp-c23 supp-c27">To quickly access data at given location and zoom level, it needs to be stored at multiple resolutions in a format that facilitates fast random access. For our application, we used a multi-resolution version of the Cooler format (</span><span class="supp-c0"><a class="supp-c9" href="https://github.com/mirnylab/cooler">https://github.com/mirnylab/cooler</a></span><span class="supp-c23 supp-c27">) to store HiC data as well as a custom hdf5-based format we called &lsquo;hitile&rsquo; for 1D data (</span><span class="supp-c0 supp-c23"><a class="supp-c9" href="http://github.com/hms-dbmi/clodius">http://github.com/hms-dbmi/clodius</a></span><span class="supp-c23 supp-c27">). Both provide fast access and compressed storage of large data sets.</span></p>
   <h3 class="supp-c15" id="h.i17w33o3h3ll"><span class="supp-c34">Data Processing</span></h3>
   <p class="supp-c1"><span>The Hi-C data used in this study was generated as described in Schwarzer et al. Hi-C data was binned using the </span><span class="supp-c8">cooler</span><span>&nbsp;Python package (</span><span class="supp-c0"><a class="supp-c9" href="https://github.com/mirnylab/cooler">https://github.com/mirnylab/cooler</a></span><span>) to produce binary Cooler (.cool) files. The Cooler format is a realization of a sparse matrix storage schema for genomic interaction matrices implemented using HDF5 (</span><span>HDF5 Group 1997-2017</span><span>), a scientific data container format that supports features including chunking, compression and random array access. The details of the Cooler format and </span><span class="supp-c8">cooler</span><span>&nbsp;package are described at </span><span class="supp-c0"><a class="supp-c9" href="http://cooler.readthedocs.io">http://cooler.readthedocs.io</a></span><span class="supp-c3">.</span></p>
   <p class="supp-c1 supp-c24"><span class="supp-c3"></span></p>
   <p class="supp-c1"><span>Starting with Cooler files containing single Hi-C matrices binned at 1 kb resolution, we use the </span><span class="supp-c32">coarsegrain</span><span>&nbsp;command from the </span><span class="supp-c8">cooler</span><span>&nbsp;package, which recursively rebins the chromosomal blocks in 2x2 pixel groupings to generate Cooler files containing contact matrices of several resolutions, starting from the input matrix (at the &ldquo;base&rdquo; resolution) up until a maximally coarsegrained matrix (the first aggregation to result in a genomewide matrix of 256 x 256 pixels or less). Each of the resulting contact matrices was normalized by computing a vector of balancing weights using a sparse implementation of the iterative correction or matrix balancing approach </span><span class="supp-c12">(Imakaev et al., 2012)</span><span class="supp-c3">, which is stored as part of the genomic bin data of the corresponding contact matrix in the Cooler file. </span></p>
   <p class="supp-c1 supp-c24"><span class="supp-c3"></span></p>
   <p class="supp-c1"><span>The resulting multiresolution files are loaded onto the higlass server as described in the higlass documentation (</span><span class="supp-c0"><a class="supp-c9" href="http://github.com/hms-dbmi/higlass/wiki">http://github.com/hms-dbmi/higlass/wiki</a></span><span class="supp-c3">). </span></p>
   <h3 class="supp-c15" id="h.5yafxk39x9mq"><span class="supp-c34">View Composition and Sharing</span></h3>
   <p class="supp-c1"><span class="supp-c3">The user interface for HiGlass is built to display multiple datasets and to facilitate their exploration through the synchronization of navigation. Supplementary Fig. 1 provides an overview of the terms we use to describe elements of the user interface. Broadly, views are individual units that can be navigated (zoomed and panned) independently or in concert with other views. Tracks are subsections of views that are always synchronized along one or two of their axes. Links describe relationships between the different views and serve to enforce synchronized navigation. Viewport mirroring displays the extent of one view in the coordinate system of another.</span></p>
   <p class="supp-c1 supp-c24"><span class="supp-c3"></span></p>
   <p class="supp-c1"><span>The view composition, or spatial arrangement and linking between views and tracks, can be configured using the user interface (</span><span class="supp-c0"><a class="supp-c9" href="https://github.com/hms-dbmi/higlass/wiki/View-operations">https://github.com/hms-dbmi/higlass/wiki/View-operations</a></span><span>) or defined in an external JSON file. Compositions created through the user interface can be exported as JSON configuration files to be shared and reproduced without having to move large data files. There is also an option to store a JSON configuration file on the server and access it through a hyperlink (</span><span class="supp-c0"><a class="supp-c9" href="https://github.com/hms-dbmi/higlass/wiki/Exporting-and-sharing">https://github.com/hms-dbmi/higlass/wiki/Exporting-and-sharing</a></span><span class="supp-c3">).</span></p>
   <h2 class="supp-c21" id="h.3io4abr4zqto"><span class="supp-c14 supp-c12">Data Availability</span></h2>
   <p class="supp-c7"><span class="supp-c3">All the data used in this paper is available to reviewers at:</span></p>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <p class="supp-c7"><span class="supp-c0"><a class="supp-c9" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=avolmkiwpjgrjex&acc=GSE93431">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=avolmkiwpjgrjex&acc=GSE93431</a></span><span class="supp-c3">&nbsp; </span></p>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <p class="supp-c7"><span class="supp-c3">Information about processing and importing into it higlass is available on the wiki:</span></p>
   <p class="supp-c4"><span class="supp-c3"></span></p>
   <p class="supp-c7"><span class="supp-c0"><a class="supp-c9" href="https://github.com/hms-dbmi/higlass/wiki">https://github.com/hms-dbmi/higlass/wiki</a></span><span>&nbsp;</span></p>
   <h2 class="supp-c21" id="h.nx1aibv30l7o"><span class="supp-c14 supp-c12">Supplementary References</span></h2>
   <p class="supp-c25"><span>Imakaev, M., Fudenberg, G., McCord, R.P., Naumova, N., Goloborodko, A., Lajoie, B.R., Dekker, J., and Mirny, L.A. (2012). Iterative correction of Hi-C data reveals hallmarks of chromosome organization. Nat. Methods</span><span class="supp-c8">9</span><span>, 999&ndash;1003</span></p>
   <p class="supp-c5"><span class="supp-c3"></span></p>
   <div>
      <p class="supp-c17 supp-c24"><span class="supp-c3"></span></p>
   </div>
   
</body>
    <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/4.8.1/pixi.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/react-bootstrap/0.32.1/react-bootstrap.min.js"></script>

    <script src='https://unpkg.com/higlass@0.8.8/dist/scripts/hglib.js'></script>

    <script>
        var viewConfFig1 = 
        {
  "zoomFixed": false,
  "views": [
    {
      "layout": {
        "w": 6,
        "h": 7,
        "x": 0,
        "y": 6,
        "i": "CxSCeaTbQbmmz2_3MX23Sw",
        "moved": false,
        "static": false
      },
      "uid": "CxSCeaTbQbmmz2_3MX23Sw",
      "initialYDomain": [
        1960265758.2450886,
        1982820664.6286192
      ],
      "autocompleteSource": "http://higlass.io/api/v1/suggest/?d=OHJakQICQD6gTD7skx4EWA&",
      "initialXDomain": [
        1955270083.4673042,
        1983650429.2479057
      ],
      "tracks": {
        "bottom": [],
        "top": [
          {
            "orientation": "1d-horizontal",
            "name": "Chromosome Axis (mm9)",
            "tilesetUid": "cGSJELSNRyOaExsJL0KejQ",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "width": 190,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "top",
            "height": 23,
            "type": "horizontal-chromosome-labels",
            "options": {},
            "uid": "EsCrHgpLQoSO8fJo8PX-9w"
          },
          {
            "name": "Gene Annotations (mm9)",
            "tilesetUid": "GUm5aBiLRCyz2PsBea7Yzg",
            "server": "http://higlass.io/api/v1",
            "width": 190,
            "position": "top",
            "height": 49,
            "type": "horizontal-gene-annotations",
            "options": {
              "labelColor": "black",
              "plusStrandColor": "black",
              "minusStrandColor": "grey",
              "labelPosition": "hidden",
              "name": "Gene Annotations (mm9)"
            },
            "uid": "TrVU7C7cTuGdf6rqqOqVsA"
          },
          {
            "name": "Schwarzer et al (2016) WT H3K27ac",
            "tilesetUid": "MnHp30AcSGmUf36zPaf1KA",
            "server": "http://higlass.io/api/v1",
            "width": 462,
            "position": "top",
            "height": 31,
            "type": "horizontal-line",
            "options": {
              "name": "WT H3K27ac",
              "valueScaling": "linear",
              "lineStrokeColor": "green",
              "labelPosition": "topLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "uid": "H3adAzSXQCS1ejqu_etcXA"
          },
          {
            "name": "Schwarzer et al (2016) WT H3K4me3",
            "tilesetUid": "EsZnlyoZQzenXHfeoxLsPw",
            "server": "http://higlass.io/api/v1",
            "width": 462,
            "position": "top",
            "height": 25,
            "type": "horizontal-line",
            "options": {
              "name": "WT H3K4me3",
              "valueScaling": "linear",
              "lineStrokeColor": "grey",
              "labelPosition": "topLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
               "labelTextOpacity": 0.2

            },
            "uid": "a0FwXQAzTY-wuIA0VOCPPQ"
          }
        ],
        "right": [
          {
            "name": "Schwarzer et al (2016) WT H3K27ac",
            "tilesetUid": "MnHp30AcSGmUf36zPaf1KA",
            "server": "http://higlass.io/api/v1",
            "width": 28,
            "position": "right",
            "height": 345,
            "type": "vertical-line",
            "options": {
              "name": "WT H3K27ac",
              "valueScaling": "linear",
              "lineStrokeColor": "green",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "uid": "Lza_OzzQReSm-hIoTTk2fA"
          },
          {
            "name": "Schwarzer et al (2016) WT H3K4me3",
            "tilesetUid": "EsZnlyoZQzenXHfeoxLsPw",
            "server": "http://higlass.io/api/v1",
            "width": 22,
            "position": "right",
            "height": 345,
            "type": "vertical-line",
            "options": {
              "name": "WT H3K4me3",
              "valueScaling": "linear",
              "lineStrokeColor": "grey",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "uid": "K3bc4N4mSu2cVOmR1PHTwQ"
          }
        ],
        "center": [
          {
            "position": "center",
            "type": "combined",
            "uid": "NxjqAn6FS4OTeTWslGK2cw",
            "contents": [
              {
                "uid": "BmYR9dxuR9WP6nGeWQoXug",
                "tilesetUid": "cTc3lt_7StSgcrr1kHWidQ",
                "server": "http://higlass.io/api/v1",
                "width": 20,
                "position": "center",
                "height": 20,
                "type": "heatmap",
                "options": {
                  "maxZoom": null,
                  "labelPosition": "bottomRight",
                  "colorRange": [
                    "white",
                    "rgba(245,166,35,1.0)",
                    "rgba(208,2,27,1.0)",
                    "black"
                  ],
                  "name": "WT",
                  "colorbarLabelsPosition": "inside",
                  "colorbarPosition": "topRight"
                },
                "name": "Schwarzer et al (2016) WT"
              }
            ],
            "options": {}
          }
        ],
        "left": [
          {
            "orientation": "1d-vertical",
            "name": "Chromosome Axis (mm9)",
            "tilesetUid": "BmjsajIaREq4P3lXQGsmXw",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "minWidth": 30,
            "width": 30,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "left",
            "height": 30,
            "type": "vertical-chromosome-labels",
            "options": {},
            "uid": "NlkjnKsoSGmfsrWKRfgBYA"
          }
        ]
      },
      "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
      "genomePositionSearchBoxVisible": false
    },
    {
      "layout": {
        "w": 6,
        "h": 7,
        "x": 6,
        "y": 6,
        "i": "KrP5zcaVSxqCj6mEI8WA2A",
        "moved": false,
        "static": false
      },
      "uid": "KrP5zcaVSxqCj6mEI8WA2A",
      "initialYDomain": [
        1960489813.6065063,
        1982596609.2672012
      ],
      "autocompleteSource": "http://higlass.io/api/v1/suggest/?d=OHJakQICQD6gTD7skx4EWA&",
      "initialXDomain": [
        1954747287.6239667,
        1984173225.091243
      ],
      "tracks": {
        "bottom": [],
        "top": [
          {
            "orientation": "1d-horizontal",
            "name": "Chromosome Axis (mm19)",
            "tilesetUid": "W1N2NIDPRZSckKMlHAKzwA",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "width": 189,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "top",
            "height": 23,
            "type": "horizontal-chromosome-labels",
            "options": {},
            "uid": "AWcBqpmASpmA01tpfgwWOg"
          },
          {
            "name": "Gene Annotations (mm9)",
            "tilesetUid": "GUm5aBiLRCyz2PsBea7Yzg",
            "server": "http://higlass.io/api/v1",
            "width": 189,
            "position": "top",
            "height": 50,
            "type": "horizontal-gene-annotations",
            "options": {
              "labelColor": "black",
              "plusStrandColor": "black",
              "minusStrandColor": "grey",
              "labelPosition": "hidden",
              "name": "Gene Annotations (mm9)"
            },
            "uid": "CNJmjwCWS--luAsiVZPO1A"
          },
          {
            "name": "Schwarzer et al (2016) NIPBL H3K27ac",
            "tilesetUid": "YtyvhRBuQoChw4hq7xxnlA",
            "server": "http://higlass.io/api/v1",
            "width": 551,
            "position": "top",
            "height": 31,
            "type": "horizontal-line",
            "options": {
              "name": "NIPBL H3K27ac",
              "valueScaling": "linear",
              "lineStrokeColor": "green",
              "labelPosition": "topLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2

            },
            "uid": "OXr0hmySTCS1aQGZOhG-sg"
          },
          {
            "name": "Schwarzer et al (2016) NIPBL H3K4me3",
            "tilesetUid": "LeW3Ri4aQSevDwVgEzGu0A",
            "server": "http://higlass.io/api/v1",
            "width": 472,
            "position": "top",
            "height": 27,
            "type": "horizontal-line",
            "options": {
              "name": "NIPBL H3K4me3",
              "valueScaling": "linear",
              "lineStrokeColor": "grey",
              "labelPosition": "topLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "uid": "RbiifZcBSwyojs8DOrlB3g"
          }
        ],
        "right": [],
        "center": [
          {
            "position": "center",
            "type": "combined",
            "uid": "fE84rvqYRWeSoiaSm-AoRg",
            "contents": [
              {
                "name": "Schwarzer et al (2016) NIPBL",
                "tilesetUid": "J2IoKaXJSG-5bH1pITTtTg",
                "server": "http://higlass.io/api/v1",
                "width": 20,
                "position": "center",
                "height": 20,
                "type": "heatmap",
                "options": {
                  "maxZoom": null,
                  "labelPosition": "bottomRight",
                  "colorRange": [
                    "white",
                    "rgba(245,166,35,1.0)",
                    "rgba(208,2,27,1.0)",
                    "black"
                  ],
                  "name": "NIPBL",
                  "colorbarLabelsPosition": "inside",
                  "colorbarPosition": "topRight"
                },
                "uid": "C-qiOuzdQhmkDB_5nvugpA"
              }
            ],
            "options": {}
          }
        ],
        "left": [
          {
            "orientation": "1d-vertical",
            "name": "Chromosome Axis (mm9)",
            "tilesetUid": "WeUIOO4oQO6FO6g2PWPhnw",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "minWidth": 30,
            "width": 22,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "left",
            "height": 148,
            "type": "vertical-chromosome-labels",
            "options": {},
            "uid": "UqH0mt73TqSebs_nFzqHvw"
          },
          {
            "name": "Schwarzer et al (2016) NIPBL H3K27ac",
            "tilesetUid": "YtyvhRBuQoChw4hq7xxnlA",
            "server": "http://higlass.io/api/v1",
            "width": 27,
            "position": "left",
            "height": 338,
            "type": "vertical-line",
            "options": {
              "name": "NIPBL H3K27ac",
              "valueScaling": "linear",
              "lineStrokeColor": "green",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "uid": "LLf91rc9T3aYRXWkDrqwEQ"
          },
          {
            "name": "Schwarzer et al (2016) NIPBL H3K4me3",
            "tilesetUid": "LeW3Ri4aQSevDwVgEzGu0A",
            "server": "http://higlass.io/api/v1",
            "width": 24,
            "position": "left",
            "height": 338,
            "type": "vertical-line",
            "options": {
              "name": "NIPBL H3K4me3",
              "valueScaling": "linear",
              "lineStrokeColor": "grey",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "uid": "LP7bzdrzTzyngJ7zig_TgA"
          }
        ]
      },
      "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
      "genomePositionSearchBoxVisible": false
    },
    {
      "layout": {
        "w": 6,
        "h": 6,
        "x": 0,
        "y": 0,
        "i": "eJLMGjH7Qm-zm_F_hqjXkw",
        "moved": false,
        "static": false
      },
      "uid": "eJLMGjH7Qm-zm_F_hqjXkw",
      "initialYDomain": [
        1947384820.6347494,
        2016972678.9517794
      ],
      "autocompleteSource": "http://higlass.io/api/v1/suggest/?d=OHJakQICQD6gTD7skx4EWA&",
      "initialXDomain": [
        1917162022.8758364,
        2017770974.6594946
      ],
      "tracks": {
        "bottom": [],
        "top": [
          {
            "name": "Gene Annotations (mm9)",
            "tilesetUid": "GUm5aBiLRCyz2PsBea7Yzg",
            "server": "http://higlass.io/api/v1",
            "width": 240,
            "position": "top",
            "height": 41,
            "type": "horizontal-gene-annotations",
            "options": {
              "labelColor": "black",
              "plusStrandColor": "black",
              "minusStrandColor": "grey",
              "labelPosition": "hidden",
              "name": "Gene Annotations (mm9)"
            },
            "uid": "M2YZ4JBQSWS0rR--oiDKhA"
          },
          {
            "orientation": "1d-horizontal",
            "name": "Chromosome Axis (mm9)",
            "tilesetUid": "cGSJELSNRyOaExsJL0KejQ",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "width": 240,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "top",
            "height": 26,
            "type": "horizontal-chromosome-labels",
            "options": {},
            "uid": "EsCrHgpLQoSO8fJo8PX-9w"
          }
        ],
        "right": [],
        "center": [
          {
            "position": "center",
            "type": "combined",
            "uid": "Dbaw-ZX2RMqOx_1ogCGA8Q",
            "contents": [
              {
                "uid": "e8OrP0OmRoiEb434kSoypg",
                "tilesetUid": "cTc3lt_7StSgcrr1kHWidQ",
                "server": "http://higlass.io/api/v1",
                "width": 20,
                "position": "center",
                "height": 20,
                "type": "heatmap",
                "options": {
                  "maxZoom": null,
                  "labelPosition": "bottomRight",
                  "colorRange": [
                    "white",
                    "rgba(245,166,35,1.0)",
                    "rgba(208,2,27,1.0)",
                    "black"
                  ],
                  "name": "WT",
                  "colorbarLabelsPosition": "inside",
                  "colorbarPosition": "topRight"
                },
                "name": "Schwarzer et al (2016) WT"
              },
              {
                "fromViewUid": "CxSCeaTbQbmmz2_3MX23Sw",
                "uid": "DQ8YUWtcQqWdep86bJ84HQ",
                "position": "center",
                "type": "viewport-projection-center",
                "options": {
                  "projectionFillOpacity": 0.3,
                  "projectionStrokeColor": "#777",
                  "projectionFillColor": "#777",
                  "projectionStrokeOpacity": 0.3
                },
                "name": "Viewport Projection"
              }
            ],
            "options": {}
          }
        ],
        "left": [
          {
            "orientation": "1d-vertical",
            "name": "Chromosome Axis (mm9)",
            "tilesetUid": "fs487zYMSMG5XLPXsakQ3g",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "minWidth": 30,
            "width": 30,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "left",
            "height": 30,
            "type": "vertical-chromosome-labels",
            "options": {},
            "uid": "aiYnl3phRPCMfwyKxKaSow"
          }
        ]
      },
      "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
      "genomePositionSearchBoxVisible": false
    },
    {
      "layout": {
        "w": 6,
        "h": 6,
        "x": 6,
        "y": 0,
        "i": "BI8YmooFSCuI9zluz_ehnA",
        "moved": false,
        "static": false
      },
      "uid": "BI8YmooFSCuI9zluz_ehnA",
      "initialYDomain": [
        1948432830.5491662,
        2015924669.0373623
      ],
      "autocompleteSource": "http://higlass.io/api/v1/suggest/?d=OHJakQICQD6gTD7skx4EWA&",
      "initialXDomain": [
        1915904410.9785466,
        2019028586.5567842
      ],
      "tracks": {
        "bottom": [],
        "top": [
          {
            "name": "Gene Annotations (mm9)",
            "tilesetUid": "GUm5aBiLRCyz2PsBea7Yzg",
            "server": "http://higlass.io/api/v1",
            "width": 240,
            "position": "top",
            "height": 48,
            "type": "horizontal-gene-annotations",
            "options": {
              "labelColor": "black",
              "plusStrandColor": "black",
              "minusStrandColor": "grey",
              "labelPosition": "hidden",
              "name": "Gene Annotations (mm9)"
            },
            "uid": "Fw793pB8RGOBLJETFtQLsQ"
          },
          {
            "orientation": "1d-horizontal",
            "name": "Chromosome Axis (mm19)",
            "tilesetUid": "W1N2NIDPRZSckKMlHAKzwA",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "width": 240,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "top",
            "height": 24,
            "type": "horizontal-chromosome-labels",
            "options": {},
            "uid": "AWcBqpmASpmA01tpfgwWOg"
          }
        ],
        "right": [],
        "center": [
          {
            "position": "center",
            "type": "combined",
            "uid": "fE84rvqYRWeSoiaSm-AoRg",
            "contents": [
              {
                "name": "Schwarzer et al (2016) NIPBL",
                "tilesetUid": "J2IoKaXJSG-5bH1pITTtTg",
                "server": "http://higlass.io/api/v1",
                "width": 20,
                "position": "center",
                "height": 20,
                "type": "heatmap",
                "options": {
                  "maxZoom": null,
                  "labelPosition": "bottomRight",
                  "colorRange": [
                    "white",
                    "rgba(245,166,35,1.0)",
                    "rgba(208,2,27,1.0)",
                    "black"
                  ],
                  "name": "NIPBL",
                  "colorbarLabelsPosition": "inside",
                  "colorbarPosition": "topRight"
                },
                "uid": "C-qiOuzdQhmkDB_5nvugpA"
              },
              {
                "fromViewUid": "KrP5zcaVSxqCj6mEI8WA2A",
                "uid": "aO17E_ZNRXuO4VfFTG-4mQ",
                "position": "center",
                "type": "viewport-projection-center",
                "options": {
                  "projectionFillOpacity": 0.3,
                  "projectionStrokeColor": "#777",
                  "projectionFillColor": "black",
                  "projectionStrokeOpacity": 0.3
                },
                "name": "Viewport Projection"
              }
            ],
            "options": {}
          }
        ],
        "left": [
          {
            "orientation": "1d-vertical",
            "name": "Chromosome Axis (mm9)",
            "tilesetUid": "Wf4p2PxkQNqy5EC5RzhanQ",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "minWidth": 30,
            "width": 24,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "left",
            "height": 161,
            "type": "vertical-chromosome-labels",
            "options": {},
            "uid": "XRk4dwN7QGyMpfvd06k1bA"
          }
        ]
      },
      "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
      "genomePositionSearchBoxVisible": false
    }
  ],
  "editable": true,
  "exportViewUrl": "http://higlass.io/api/v1/viewconfs",
  "zoomLocks": {
    "locksByViewUid": {
      "eJLMGjH7Qm-zm_F_hqjXkw": "fnM1HM2fT8KXb4w_0gKRcA",
      "KrP5zcaVSxqCj6mEI8WA2A": "JE-pqvTZRQWVFsN0AmP75w",
      "CxSCeaTbQbmmz2_3MX23Sw": "JE-pqvTZRQWVFsN0AmP75w",
      "BI8YmooFSCuI9zluz_ehnA": "fnM1HM2fT8KXb4w_0gKRcA"
    },
    "locksDict": {
      "fnM1HM2fT8KXb4w_0gKRcA": {
        "eJLMGjH7Qm-zm_F_hqjXkw": [
          1966916755.7812533,
          1972013566.595924,
          79705.25699186325
        ],
        "BI8YmooFSCuI9zluz_ehnA": [
          1966916755.7812533,
          1972013566.595924,
          79705.25699186325
        ]
      },
      "JE-pqvTZRQWVFsN0AmP75w": {
        "KrP5zcaVSxqCj6mEI8WA2A": [
          1974905964.0722282,
          1957723677.693444,
          406590.3164639473
        ],
        "CxSCeaTbQbmmz2_3MX23Sw": [
          1974905964.0722282,
          1957723677.693444,
          406590.3164639473
        ]
      }
    }
  },
  "trackSourceServers": [
    "http://higlass.io/api/v1"
  ],
  "locationLocks": {
    "locksByViewUid": {
      "aa": "e2zEAzWbRFGFdfBcf3GMAg",
      "Kq_XUAkFQjmMjoMJ8ocYNA": "aNaNY94gSu-XA4W2Ya6cag",
      "eJLMGjH7Qm-zm_F_hqjXkw": "aNaNY94gSu-XA4W2Ya6cag",
      "CxSCeaTbQbmmz2_3MX23Sw": "e2zEAzWbRFGFdfBcf3GMAg",
      "BI8YmooFSCuI9zluz_ehnA": "aNaNY94gSu-XA4W2Ya6cag",
      "KrP5zcaVSxqCj6mEI8WA2A": "e2zEAzWbRFGFdfBcf3GMAg"
    },
    "locksDict": {
      "e2zEAzWbRFGFdfBcf3GMAg": {
        "aa": [
          1974905964.0722282,
          1957723677.6934438,
          406590.3164639473
        ],
        "KrP5zcaVSxqCj6mEI8WA2A": [
          1974905964.0722282,
          1957723677.693444,
          406590.3164639473
        ],
        "CxSCeaTbQbmmz2_3MX23Sw": [
          1974905964.0722282,
          1957723677.693444,
          406590.3164639473
        ]
      },
      "aNaNY94gSu-XA4W2Ya6cag": {
        "eJLMGjH7Qm-zm_F_hqjXkw": [
          1966916755.7812533,
          1972013566.595924,
          79705.25699186325
        ],
        "Kq_XUAkFQjmMjoMJ8ocYNA": [
          1966916755.7812533,
          1972013566.595924,
          79705.25699186325
        ],
        "BI8YmooFSCuI9zluz_ehnA": [
          1966916755.7812533,
          1972013566.595924,
          79705.25699186325
        ]
      }
    }
  }
}

var viewConfFig2 =
{
  "zoomFixed": false,
  "views": [
    {
      "layout": {
        "w": 6,
        "h": 12,
        "x": 0,
        "y": 0,
        "i": "aa",
        "moved": false,
        "static": false
      },
      "uid": "aa",
      "initialYDomain": [
        2000168734.7116277,
        2002843005.743648
      ],
      "autocompleteSource": "http://higlass.io/api/v1/suggest/?d=OHJakQICQD6gTD7skx4EWA&",
      "initialXDomain": [
        1965252860.578572,
        1967427064.6696455
      ],
      "tracks": {
        "bottom": [],
        "top": [
          {
            "uid": "LPS8hy9ESTmRH84egjz5ug",
            "tilesetUid": "GUm5aBiLRCyz2PsBea7Yzg",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "top",
            "height": 60,
            "type": "horizontal-gene-annotations",
            "options": {
              "labelColor": "black",
              "plusStrandColor": "black",
              "minusStrandColor": "grey",
              "labelPosition": "hidden",
              "name": "Gene Annotations (mm9)"
            },
            "name": "Gene Annotations (mm9)"
          },
          {
            "uid": "c1lqPpfYQmG7zODVg2RREA",
            "tilesetUid": "M-LTWpoGQ0iQvfJBtzXg4A",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "top",
            "height": 20,
            "type": "horizontal-line",
            "options": {
              "name": "WT RNAseq (+ strand)",
              "valueScaling": "log",
              "lineStrokeColor": "brown",
              "labelPosition": "outerLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) WT RNAseq (+ strand)"
          },
          {
            "uid": "dPdjUPXdSPuVLheuE3fQtg",
            "tilesetUid": "Ch2IpAcXR12cp4aRFfDJyw",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "top",
            "height": 20,
            "type": "horizontal-line",
            "options": {
              "name": "WT RNAseq (- strand)",
              "valueScaling": "log",
              "lineStrokeColor": "blue",
              "labelPosition": "outerLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) WT RNAseq (- strand)"
          },
          {
            "uid": "E2_UjX31Slaiwbp20crHXw",
            "tilesetUid": "MnHp30AcSGmUf36zPaf1KA",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "top",
            "height": 20,
            "type": "horizontal-line",
            "options": {
              "name": "WT H3K27ac",
              "valueScaling": "linear",
              "lineStrokeColor": "green",
              "labelPosition": "outerLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) WT H3K27ac"
          },
          {
            "uid": "Lfqhi0l7RWCjIK5K-Lwufw",
            "tilesetUid": "EsZnlyoZQzenXHfeoxLsPw",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "top",
            "height": 20,
            "type": "horizontal-line",
            "options": {
              "name": "WT H3K4me3",
              "valueScaling": "linear",
              "lineStrokeColor": "grey",
              "labelPosition": "outerLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) WT H3K4me3"
          },
          {
            "orientation": "1d-horizontal",
            "name": "Chromosome Axis (mm9)",
            "tilesetUid": "NEm2Sv3-RIqFCLeNGr7zPQ",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "width": 20,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "top",
            "height": 30,
            "type": "horizontal-chromosome-labels",
            "options": {},
            "uid": "XHHLMTDRR5CuTt7dFRcxiw"
          }
        ],
        "right": [],
        "center": [
          {
            "position": "center",
            "type": "combined",
            "uid": "Wcf4uK2oTRi2UC8vK2U-Xg",
            "contents": [
              {
                "uid": "B4Bv8J7TTASH6ojud_TchA",
                "tilesetUid": "cTc3lt_7StSgcrr1kHWidQ",
                "server": "http://higlass.io/api/v1",
                "width": 20,
                "position": "center",
                "height": 20,
                "type": "heatmap",
                "options": {
                  "maxZoom": "9",
                  "labelPosition": "bottomRight",
                  "colorRange": [
                    "white",
                    "rgba(245,166,35,1.0)",
                    "rgba(208,2,27,1.0)",
                    "black"
                  ],
                  "name": "WT",
                  "colorbarLabelsPosition": "inside",
                  "colorbarPosition": "hidden"
                },
                "name": "Schwarzer et al (2016) WT"
              }
            ],
            "options": {}
          }
        ],
        "left": [
          {
            "uid": "bcPduIxqRdK7hFl5kYFR1g",
            "tilesetUid": "GUm5aBiLRCyz2PsBea7Yzg",
            "server": "http://higlass.io/api/v1",
            "width": 60,
            "position": "left",
            "height": 20,
            "type": "vertical-gene-annotations",
            "options": {
              "labelColor": "black",
              "plusStrandColor": "black",
              "minusStrandColor": "grey",
              "labelPosition": "hidden",
              "name": "Gene Annotations (mm9)"
            },
            "name": "Gene Annotations (mm9)"
          },
          {
            "uid": "OlOaB7VVSLiaqxpopiQAzw",
            "tilesetUid": "M-LTWpoGQ0iQvfJBtzXg4A",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "left",
            "height": 20,
            "type": "vertical-line",
            "options": {
              "name": "WT RNAseq (+ strand)",
              "valueScaling": "log",
              "lineStrokeColor": "brown",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) WT RNAseq (+ strand)"
          },
          {
            "uid": "ambHQA5WSXWce9ZFjWwq_g",
            "tilesetUid": "Ch2IpAcXR12cp4aRFfDJyw",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "left",
            "height": 20,
            "type": "vertical-line",
            "options": {
              "name": "WT RNAseq (- strand)",
              "valueScaling": "log",
              "lineStrokeColor": "blue",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) WT RNAseq (- strand)"
          },
          {
            "uid": "D_7k_3PGS_e6XxKcARJ-pg",
            "tilesetUid": "MnHp30AcSGmUf36zPaf1KA",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "left",
            "height": 20,
            "type": "vertical-line",
            "options": {
              "name": "WT H3K27ac",
              "valueScaling": "linear",
              "lineStrokeColor": "green",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) WT H3K27ac"
          },
          {
            "uid": "U2-yGyo1TBaHyNR0Tq5LPA",
            "tilesetUid": "EsZnlyoZQzenXHfeoxLsPw",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "left",
            "height": 20,
            "type": "vertical-line",
            "options": {
              "name": "WT H3K4me3",
              "valueScaling": "linear",
              "lineStrokeColor": "grey",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) WT H3K4me3"
          },
          {
            "orientation": "1d-vertical",
            "name": "Chromosome Axis (mm9)",
            "tilesetUid": "A_seqdp2TE-PNOaDKcF_eg",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "minWidth": 30,
            "width": 30,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "left",
            "height": 30,
            "type": "vertical-chromosome-labels",
            "options": {},
            "uid": "K2cqfrYCS4CizVhSSGwSjA"
          }
        ]
      },
      "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
      "genomePositionSearchBoxVisible": false
    },
    {
      "layout": {
        "w": 12,
        "h": 9,
        "x": 0,
        "y": 12,
        "i": "YhIZIRwVQjmBfNfB_VOyHQ",
        "moved": false,
        "static": false
      },
      "uid": "YhIZIRwVQjmBfNfB_VOyHQ",
      "initialYDomain": [
        1994320951.9461486,
        2008159160.2773929
      ],
      "autocompleteSource": "http://higlass.io/api/v1/suggest/?d=OHJakQICQD6gTD7skx4EWA&",
      "initialXDomain": [
        1944654664.5483615,
        1985628130.557073
      ],
      "tracks": {
        "bottom": [],
        "top": [
          {
            "orientation": "1d-horizontal",
            "name": "Chromosome Axis (mm9)",
            "tilesetUid": "EIPhTdl3S9KtQWMdpX-OTw",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "width": 560,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "top",
            "height": 30,
            "type": "horizontal-chromosome-labels",
            "options": {},
            "uid": "XRRTikI4RVaqV_5Nvnpi2Q"
          }
        ],
        "right": [],
        "center": [
          {
            "position": "center",
            "type": "combined",
            "uid": "OcsQL9IdShmkmlWiTvi63Q",
            "contents": [
              {
                "uid": "HvzA4MVORgapXZ1u7JhD9Q",
                "tilesetUid": "J2IoKaXJSG-5bH1pITTtTg",
                "server": "http://higlass.io/api/v1",
                "width": 560,
                "position": "center",
                "height": 296,
                "type": "heatmap",
                "options": {
                  "maxZoom": "7",
                  "labelPosition": "bottomRight",
                  "colorRange": [
                    "white",
                    "rgba(245,166,35,1.0)",
                    "rgba(208,2,27,1.0)",
                    "black"
                  ],
                  "name": "NIPBL",
                  "colorbarLabelsPosition": "inside",
                  "colorbarPosition": "topRight"
                },
                "name": "Schwarzer et al (2016) NIPBL"
              },
              {
                "fromViewUid": "aa",
                "uid": "L5eYgpnrRmO7D3kSbeLtOQ",
                "position": "center",
                "type": "viewport-projection-center",
                "options": {
                  "projectionFillOpacity": 0.3,
                  "projectionStrokeColor": "#777",
                  "projectionFillColor": "#777",
                  "projectionStrokeOpacity": 0.3
                },
                "name": "Viewport Projection",
                "width": 560,
                "height": 296
              }
            ],
            "options": {},
            "width": 560,
            "height": 296
          }
        ],
        "left": [
          {
            "type": "vertical-chromosome-labels",
            "local": true,
            "orientation": "1d-vertical",
            "minWidth": 30,
            "minHeight": 30,
            "name": "Chromosome Axis (hg19)",
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/hg19/chromSizes.tsv",
            "thumbnail": null,
            "server": "",
            "tilesetUid": "LONDAP1pQfe7UqVi_bWZHA",
            "uid": "FmihhNKMTPmwHiSvqu2Usg",
            "options": {},
            "width": 30,
            "height": 30,
            "position": "left"
          }
        ]
      },
      "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
      "genomePositionSearchBoxVisible": false
    },
    {
      "layout": {
        "w": 6,
        "h": 12,
        "x": 6,
        "y": 0,
        "i": "P8FcBND2TdqhYuXVC5TSpw",
        "moved": false,
        "static": false
      },
      "uid": "P8FcBND2TdqhYuXVC5TSpw",
      "initialYDomain": [
        2000168734.711623,
        2002843005.743653
      ],
      "autocompleteSource": "http://higlass.io/api/v1/suggest/?d=OHJakQICQD6gTD7skx4EWA&",
      "initialXDomain": [
        1965252860.5785675,
        1967427064.6696491
      ],
      "tracks": {
        "bottom": [],
        "top": [
          {
            "uid": "VnO774sdThWwo7MeXvzHTQ",
            "tilesetUid": "GUm5aBiLRCyz2PsBea7Yzg",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "top",
            "height": 60,
            "type": "horizontal-gene-annotations",
            "options": {
              "labelColor": "black",
              "plusStrandColor": "black",
              "minusStrandColor": "grey",
              "labelPosition": "hidden",
              "name": "Gene Annotations (mm9)"
            },
            "name": "Gene Annotations (mm9)"
          },
          {
            "uid": "YnOTn7vDTmOp2S72hvbrfA",
            "tilesetUid": "LP-NPfCTQbeOJGJ8q_ar9Q",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "top",
            "height": 20,
            "type": "horizontal-line",
            "options": {
              "name": "NIPBL RNAseq (+ strand)",
              "valueScaling": "log",
              "lineStrokeColor": "brown",
              "labelPosition": "outerLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) NIPBL RNAseq (+ strand)"
          },
          {
            "uid": "BY_odP2KQX2cAy-hMSTUJg",
            "tilesetUid": "TIXmwwQqQ9aigroc6myY7Q",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "top",
            "height": 20,
            "type": "horizontal-line",
            "options": {
              "name": "NIPBL RNAseq (- strand)",
              "valueScaling": "log",
              "lineStrokeColor": "blue",
              "labelPosition": "outerLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) NIPBL RNAseq (- strand)"
          },
          {
            "uid": "cj9HpQt6QB-1pRrYnKh6kw",
            "tilesetUid": "YtyvhRBuQoChw4hq7xxnlA",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "top",
            "height": 20,
            "type": "horizontal-line",
            "options": {
              "name": "NIPBL H3K27ac",
              "valueScaling": "linear",
              "lineStrokeColor": "green",
              "labelPosition": "outerLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) NIPBL H3K27ac"
          },
          {
            "uid": "RbJwexagTZCO7Rw3GtcOCg",
            "tilesetUid": "LeW3Ri4aQSevDwVgEzGu0A",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "top",
            "height": 20,
            "type": "horizontal-line",
            "options": {
              "name": "NIPBL H3K4me3",
              "valueScaling": "linear",
              "lineStrokeColor": "grey",
              "labelPosition": "outerLeft",
              "labelColor": "black",
              "axisPositionHorizontal": "right",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) NIPBL H3K4me3"
          },
          {
            "orientation": "1d-horizontal",
            "name": "Chromosome Axis (mm9)",
            "tilesetUid": "GReSiVrRTaG8xOjigr3yMQ",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "width": 20,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "top",
            "height": 30,
            "type": "horizontal-chromosome-labels",
            "options": {},
            "uid": "eAJwJCYrR5yOa4XcJ3fMPQ"
          }
        ],
        "right": [],
        "center": [
          {
            "position": "center",
            "type": "combined",
            "uid": "OcsQL9IdShmkmlWiTvi63Q",
            "contents": [
              {
                "uid": "HvzA4MVORgapXZ1u7JhD9Q",
                "tilesetUid": "J2IoKaXJSG-5bH1pITTtTg",
                "server": "http://higlass.io/api/v1",
                "width": 20,
                "position": "center",
                "height": 20,
                "type": "heatmap",
                "options": {
                  "maxZoom": "9",
                  "labelPosition": "bottomRight",
                  "colorRange": [
                    "white",
                    "rgba(245,166,35,1.0)",
                    "rgba(208,2,27,1.0)",
                    "black"
                  ],
                  "name": "NIPBL",
                  "colorbarLabelsPosition": "inside",
                  "colorbarPosition": "hidden"
                },
                "name": "Schwarzer et al (2016) NIPBL"
              }
            ],
            "options": {}
          }
        ],
        "left": [
          {
            "uid": "GMfBK1DnQY22YvEMmD2auw",
            "tilesetUid": "GUm5aBiLRCyz2PsBea7Yzg",
            "server": "http://higlass.io/api/v1",
            "width": 60,
            "position": "left",
            "height": 20,
            "type": "vertical-gene-annotations",
            "options": {
              "labelColor": "black",
              "plusStrandColor": "black",
              "minusStrandColor": "grey",
              "labelPosition": "hidden",
              "name": "Gene Annotations (mm9)"
            },
            "name": "Gene Annotations (mm9)"
          },
          {
            "uid": "YDrkErk0T62grdGyJuvp0Q",
            "tilesetUid": "LP-NPfCTQbeOJGJ8q_ar9Q",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "left",
            "height": 20,
            "type": "vertical-line",
            "options": {
              "name": "NIPBL RNAseq (+ strand)",
              "valueScaling": "log",
              "lineStrokeColor": "brown",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) NIPBL RNAseq (+ strand)"
          },
          {
            "uid": "XvXexWQuTsu1BZTaDGWPNA",
            "tilesetUid": "TIXmwwQqQ9aigroc6myY7Q",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "left",
            "height": 20,
            "type": "vertical-line",
            "options": {
              "name": "NIPBL RNAseq (- strand)",
              "valueScaling": "log",
              "lineStrokeColor": "blue",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) NIPBL RNAseq (- strand)"
          },
          {
            "uid": "ZtTQA2IhS46manf4VZWZOQ",
            "tilesetUid": "YtyvhRBuQoChw4hq7xxnlA",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "left",
            "height": 20,
            "type": "vertical-line",
            "options": {
              "name": "NIPBL H3K27ac",
              "valueScaling": "linear",
              "lineStrokeColor": "green",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) NIPBL H3K27ac"
          },
          {
            "uid": "PjYa1AIASWaNstOn3c9big",
            "tilesetUid": "LeW3Ri4aQSevDwVgEzGu0A",
            "server": "http://higlass.io/api/v1",
            "width": 20,
            "position": "left",
            "height": 20,
            "type": "vertical-line",
            "options": {
              "name": "NIPBL H3K4me3",
              "valueScaling": "linear",
              "lineStrokeColor": "grey",
              "labelPosition": "bottomLeft",
              "axisPositionVertical": "top",
              "labelColor": "black",
              "lineStrokeWidth": 1,
              "labelTextOpacity": 0.2
            },
            "name": "Schwarzer et al (2016) NIPBL H3K4me3"
          },
          {
            "orientation": "1d-vertical",
            "name": "Chromosome Axis (mm9)",
            "tilesetUid": "A_seqdp2TE-PNOaDKcF_eg",
            "local": true,
            "minHeight": 30,
            "thumbnail": null,
            "server": "",
            "minWidth": 30,
            "width": 30,
            "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
            "position": "left",
            "height": 30,
            "type": "vertical-chromosome-labels",
            "options": {},
            "uid": "K2cqfrYCS4CizVhSSGwSjA"
          }
        ]
      },
      "chromInfoPath": "//s3.amazonaws.com/pkerp/data/mm9/chromSizes.tsv",
      "genomePositionSearchBoxVisible": false
    }
  ],
  "editable": true,
  "exportViewUrl": "http://higlass.io/api/v1/viewconfs",
  "zoomLocks": {
    "locksByViewUid": {
      "aa": "KI8WSY8FQQ2ZpiChMGrs2w",
      "P8FcBND2TdqhYuXVC5TSpw": "KI8WSY8FQQ2ZpiChMGrs2w"
    },
    "locksDict": {
      "KI8WSY8FQQ2ZpiChMGrs2w": {
        "aa": [
          1969693607.7452817,
          1975759397.1358833,
          99451.94593286514
        ],
        "P8FcBND2TdqhYuXVC5TSpw": [
          1969693607.7452812,
          1975759397.1358836,
          99451.94593286514
        ]
      }
    }
  },
  "trackSourceServers": [
    "http://higlass.io/api/v1"
  ],
  "locationLocks": {
    "locksByViewUid": {
      "aa": "QF84uirbTTewvNfR9ZyUmQ",
      "P8FcBND2TdqhYuXVC5TSpw": "QF84uirbTTewvNfR9ZyUmQ"
    },
    "locksDict": {
      "QF84uirbTTewvNfR9ZyUmQ": {
        "aa": [
          1969693607.7452817,
          1975759397.1358833,
          99451.94593286514
        ],
        "P8FcBND2TdqhYuXVC5TSpw": [
          1969693607.7452812,
          1975759397.1358836,
          99451.94593286514
        ]
      }
    }
  }
}
              window.higlassApi;

      hglib.viewer(
          document.getElementById('figure1'),
          viewConfFig1,
          { bounded: true },
          function (api) {
              window.hgApi = api;
          }
      );

            hglib.viewer(
          document.getElementById('figure2'),
          viewConfFig2,
          { bounded: true },
          function (api) {
              window.hgApi = api;
          }
      );
    </script>
</html>
